---
output: html_document
editor_options: 
  chunk_output_type: console
---
# ROSMAP
```{r, warning=F, message=F, error=F, echo=FALSE}
# library(knitr)
# library(kableExtra)
library(tidyverse)
library(janitor)
library(gtsummary)
library(dlookr)
library(inspectdf)
library(skimr)
knitr::opts_knit$set(root.dir = '/sc/arion/projects/LOAD/shea/Projects/mtDNAcn')
# setwd('/sc/arion/projects/LOAD/shea/Projects/mtDNAcn')
```

The samples that we have profiled come from two prospective studies of aging-The Religious order Study (ROS) and the Memory and Aging Project (MAP)-that recruit older individuals without known dementia and include (1) detailed cognitive, neuroimaging and other ante-mortem phenotyping and (2) an autopsy at the time of death that includes a structured neuropathologic examination. A subset of the ROSMAP samples (n=1200 for 1179 unique deceased participants) underwent whole genome sequencing, with DNA coming from brain tissue (n=806), whole blood (n=389) or lymphocytes transformed with EBV virus (n=5) (@10.1038/sdata.2018.142).

Data Dictionaries for ROSMAP can be found at: 

- [AMP-AD](https://adknowledgeportal.synapse.org/Explore/Studies?Study=syn3219045)
- [RADC](https://www.radc.rush.edu/docs/var/variables.htm)

```{r load-rosmap, warning=F, message=F, error=F}
## Phenotypic data
rosmap.pheno_bl <- readxl::read_xlsx("data/AMPAD_extra/rosmap/dataset_899_basic_08-22-2020.xlsx") %>%
  mutate(projid = as.numeric(projid))
rosmap.pheno_long.raw <- readxl::read_xlsx("data/AMPAD_extra/rosmap/dataset_899_long_08-22-2020.xlsx") %>%
  mutate(projid = as.numeric(projid))

# wgs_clinical.raw <- read_csv('data/AMPAD_extra/rosmap/ROSMAP_Clinical_2019-05_v3.csv')
## Mitochondrial 
rosmap.wgsqc <- read_csv("data/AMPAD_extra/rosmap/WGS_sample_QC_info.csv", guess_max = 10000)
mosdepth <- read_tsv("data/mosdepth/mosdepth_mtDNAcn_All.txt")
haplogrep <- read_tsv("data/haplogrep/haplogrep_jointAll.txt")

vcf <- read.vcfR("data/AMPAD_extra/rosmap/DEJ_11898_B01_GRM_WGS_2017-05-15_APOE.recalibrated_variants.vcf.gz")
```


```{r munge-rosmap, warning=F, message=F, error=F}
Z <- vcfR2tidy(vcf, format_fields = c("GT"))
apoe <- Z$gt %>% 
  filter(POS %in% c(45412079, 45411941)) %>% 
  mutate(snp = as_factor(POS), 
         snp = fct_recode(snp, "rs7412" = "45412079", "rs429358" = "45411941")) %>%
  select(Indiv, snp, gt_GT_alleles) %>% 
  pivot_wider(values_from = gt_GT_alleles, names_from = snp) %>% 
    mutate(
      apoe = case_when(
          rs429358 == "T/C" & rs7412 == "C/T" ~ 24,
          rs429358 == "T/T" & rs7412 == "T/T" ~ 22,
          rs429358 == "T/T" & rs7412 == "C/T" ~ 23,
          rs429358 == "T/T" & rs7412 == "C/C" ~ 33,
          rs429358 == "T/C" & rs7412 == "C/C" ~ 34,
          rs429358 == "C/C" & rs7412 == "C/C" ~ 44
        )
    ) %>% 
  left_join(select(rosmap.wgsqc, projid, WGS_id), by = c('Indiv' = 'WGS_id')) %>% 
  distinct(projid, .keep_all = TRUE) %>% 
  select(-Indiv)

## extract baseline and last vist values and spread
rosmap.pheno_long <- rosmap.pheno_long.raw %>% 
  group_by(projid) %>%
  slice(c(which.min(fu_year), which.max(fu_year))) %>%
  mutate(visit = case_when(fu_year == min(fu_year) ~ "bl", 
                           fu_year == max(fu_year) ~ "lv")) %>%
  distinct(., fu_year, .keep_all = TRUE) %>%
  ungroup() %>%
  pivot_wider(names_from = visit, values_from = c(-projid, -study, -scaled_to))

## Merge WGS files 
rosmap_wgs <- rosmap.wgsqc %>% 
  select(projid, SampleID = WGS_id, Source.Tissue.Type, QC) %>% 
  left_join(mosdepth, by = "SampleID") %>% 
  left_join(haplogrep, by = "SampleID") %>% 
  filter(QC == "Pass") %>% 
  mutate(macro = case_when(
    str_detect(Haplogroup, "^L|^HV|^JT") ~ substr(Haplogroup, start = 1, stop = 2), 
    TRUE ~ substr(Haplogroup, start = 1, stop = 1)
  ))

### 16 duplicate IDS 
# get_dupes(rosmap_wgs, projid)  %>% 
#   print(n = Inf)

rosmap <- rosmap.pheno_bl %>%
  left_join(apoe, by = "projid") %>%
  left_join(rosmap.pheno_long, by = c("projid", "study", "scaled_to")) %>%
  left_join(rosmap_wgs, by = "projid") %>%
    mutate(race7 = as_factor(race7),
        race7 = fct_recode(race7,'White' = '1', 'Black' = '2', "AMR" = "3", "Oceania" = "4", 
                               "Asian" = "5", "Other" = "6", "Other" = "7"),
        z_mtdnacn = scale(mtcn_avg, center = TRUE, scale = TRUE)[,1],
        spanish = as_factor(spanish),
        spanish = fct_recode(spanish,'Yes' = '1', 'No' = '2'),
        organ = recode(Source.Tissue.Type, 'Blood' = 'blood', 'Blood-PBMC' = 'blood', 'Whole Blood' = 'blood', 
                       'Blood-Cerebellum' = 'brain',  'Brain-Anterior Caudate' = 'brain',
                       'Brain-Cerebellum' = 'brain', 'Brain-DLPFC' = 'brain', 
                       'Brain-Frontal Cortex (BA unknown)' = 'brain', 
                       'Brain-Frontal Pole (BA10-12,32)' = 'brain', 
                       'Brain-Occipital Association Cortex (BA18,19)' = 'brain', 
                       'Brain-PCC' = 'brain', 'Brain-Posterior Cingulate Cortex' = 'brain', 
                       'Brain-region unknown' = 'brain', 
                       'lymphocytes _transformed _with EBV virus'= 'lymphocytes'),
        organ = as_factor(organ),
        apoe_genotype = ifelse(is.na(apoe_genotype), apoe, apoe_genotype),
        apoe4 = recode(apoe_genotype, '22' = 'e4-', '23' = 'e4-', '33' = 'e4-', '24' = 'e4+', '34' = 'e4+', '44' = 'e4+'),
        tomm40_hap = as_factor(tomm40_hap),
        tomm40_hap = fct_recode(tomm40_hap, 'S/S' = '1', 'S/L' = '2', "S/VL" = "3", "L/L" = "4", "L/VL" = "5", "VL/VL" = "6"),
        aod_cat = cut(age_death, c(50, 60, 70, 80, 90, Inf), c('50-59', '60-69', '70-79', '80-89', '90+'), right = FALSE),
        aod_cat = ordered(aod_cat, levels = c('50-59', '60-69', '70-79', '80-89', '90+')), 
        msex = as.factor(msex), 
        msex = fct_recode(msex, 'M' = '1', 'F' = '0'),
        cogdx = factor(cogdx), 
        r_pd_bl = factor(r_pd_bl), 
        r_pd_lv = factor(r_pd_lv), 
        r_stroke_bl = factor(r_stroke_bl), 
        r_stroke_lv = factor(r_stroke_lv), 
        dcfdx_bl = factor(dcfdx_bl),
        dcfdx_lv = factor(dcfdx_lv),
        apoe_genotype = as.factor(apoe_genotype), 
        apoe4 = as.factor(apoe4), 
        study = as.factor(study), 
        braaksc = ordered(braaksc, levels = c('0', '1', '2', '3', '4', '5', '6')), 
        ceradsc = ordered(ceradsc, levels = c('4', '3', '2', '1')), 
        dlbdx = as.factor(dlbdx), 
        ci_num2_mct = as.factor(ci_num2_mct), 
        ci_num2_gct = as.factor(ci_num2_gct), 
        cvda_4gp2 = as.factor(cvda_4gp2),
        caa_4gp = as.factor(caa_4gp),
        arteriol_scler = as.factor(arteriol_scler),
         hspath_typ = as.factor(hspath_typ),
         tdp_st4 = as.factor(tdp_st4),
        niareagansc = ordered(niareagansc, levels = c('4', '3', '2', '1')), 
        CDR = cut(cts_mmse30_lv, breaks = c(-Inf, 11, 21, 26, 30, Inf), labels = c(3, 2, 1, 0.5, 0), right = FALSE)) %>%
  filter(!is.na(study)) 

write_rds(rosmap, 'output/rosmap.rds')

df <- rosmap %>% 
    select(study, age_bl, msex, educ, apoe_genotype, cogdx, age_first_ad_dx, Source.Tissue.Type) 
```

```{r rosmap_dic, echo=FALSE}

rosmap_dic <- tribble(
  ~variables, ~definitions,
  "study", "Study",
  "race7", "Racial group",
  "spanish", "Spanish ethnicity",
  "msex", "Sex",
  "educ", "Education", 
  "age_bl", "Age at baseline",
  "age_at_visit_bl", "Age at baseline", 
  "age_at_visit_lv", "Age at last visit",
  "age_death", "Age at death",
  "Source.Tissue.Type",  "Source tissue for DNA",
  "organ", "collapsed source tissue into organ", 
  "apoe_genotype", "APOE genotypes",
  "apoe4", "APOE e4 carriers",
  "dcfdx_bl", "Physician’s overall cognitive diagnostic category - baseline",
  "dcfdx_lv", "Physician’s overall cognitive diagnostic category - last visit",
  "r_pd_bl", "Diagnosis of Parkinson's disease - baseline",
  "r_pd_lv", "Diagnosis of Parkinson's disease  - last visit",
  "r_stroke_bl", "Diagnosis of stroke made by clinician - baseline",
  "r_stroke_lv", "Diagnosis of stroke made by clinician - last visit",
  "age_first_ad_dx", "Age at cycle where first Alzheimer's dementia diagnosis was given", 
  "cogdx", "Clinical consensus diagnosis of cognitive status at time of death", 
  "fu_year_bl", "follow-up year at baseline", 
  "fu_year_lv", "follow-up year at last visit",
  "tomm40_hap", "TOMM40 genotype"
      )

```

## Demographics 

Demographic variables avaliable in ROSMAP are shown in Table \@ref(tab:rosmap-demo-global).    

```{r rosmap-demo-global, echo=FALSE, message=F, tidy=FALSE}
rosmap %>% 
  select(study, race7, spanish, msex, educ, age_bl, age_death) %>% 
  diagnose()  %>% 
  mutate_if(is.numeric, round, 2) %>% 
  left_join(rosmap_dic) %>% 
  select(variables, definitions, everything(), -missing_count, -unique_rate) %>% 
  knitr::kable(caption = "Data Summary")  # %>%
#  kableExtra::kable_styling() %>%
#  kableExtra::scroll_box(width = "100%")

```

<br> Descriptive statistics of numerica varibles are presented in Table \@ref(tab:rosmap-demo-numeric). 

```{r rosmap-demo-numeric, echo=FALSE, message=FALSE, tidy=FALSE}
demo_n <- rosmap %>% 
  select(study, race7, spanish, msex, educ, age_bl, age_at_visit_lv, age_death, fu_year_bl, fu_year_lv) %>% 
  inspect_num(.) 

demo_n %>% 
  select(-hist) %>% 
  mutate_if(is.numeric, round, 2) %>%
  knitr::kable(caption = "Variable type: Numeric") 
```

<br> Frequency and proportions of categorical varibles are presented in Table \@ref(tab:rosmap-demo-factor). 

```{r rosmap-demo-factor, echo=FALSE}
demo_c <- rosmap %>% 
  select(study, race7, spanish, msex, educ, age_bl, age_at_visit_lv, age_death) %>% 
  mutate_if(is.factor, fct_lump_n, 4) %>%
  inspect_cat(.)   

demo_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  knitr::kable(caption = "Variable type: Factor") %>%
  kableExtra::kable_styling(full_width = F)
```

### Plots 

```{r}
demo_n %>% show_plot()

demo_c %>% show_plot(high_cardinality = 5)
```


## Genetics 

```{r rosmap-genetic-global, echo=FALSE, message=F, tidy=FALSE}
rosmap %>% 
  select(Source.Tissue.Type, organ, 
         apoe_genotype, apoe4, tomm40_hap) %>% 
  diagnose()  %>% 
  mutate_if(is.numeric, round, 2) %>% 
  left_join(rosmap_dic) %>% 
  select(variables, definitions, everything(), -missing_count, -unique_rate) %>% 
  knitr::kable(caption = "Data Summary")  # %>%
#  kableExtra::kable_styling() %>%
#  kableExtra::scroll_box(width = "100%")

```
<br> 

```{r rosmap-genetic-factor, echo=FALSE}
genetic_c <- rosmap %>% 
  select(Source.Tissue.Type, organ, 
         apoe_genotype, apoe4, tomm40_hap) %>% 
#  mutate_if(is.factor, fct_lump_n, 4) %>%
  mutate_if(is.character, fct_lump_n, 4) %>%
  inspect_cat(.)   

genetic_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  knitr::kable(caption = "Variable type: Factor") %>%
  kableExtra::kable_styling(full_width = F)
```

### Plots 

```{r}
genetic_c %>% show_plot(high_cardinality = 5)
```

## Mitochondria 

```{r mt-numeric, echo=FALSE}
mt_n <- rosmap %>% 
  select(autosomal_coverage, mt_coverage, mtcn_avg, Haplogroup, Quality, macro) %>% 
  inspect_num(.) 

mt_n %>% 
  select(-hist) %>% 
  mutate_if(is.numeric, round, 2) %>%
  knitr::kable(caption = "Variable type: Numeric") 

```

```{r rosmap-mt-factor, echo=FALSE}
mt_c <- rosmap %>% 
  select(Haplogroup, 
         macro) %>% 
  mutate_if(is.character, fct_lump_n, 10) %>%
  inspect_cat(.)   

mt_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  knitr::kable(caption = "Variable type: Factor") %>%
  kableExtra::kable_styling(full_width = F)
```

### Plots 

```{r mt-plots}
mt_n %>% show_plot()

mt_c %>% show_plot(high_cardinality = 5)
```

### diagnosis and mtDNAcn

```{r dx-mtdnacn}
mtcn_dx <- rosmap %>%
    filter(!is.na(mtcn_avg)) %>%
    mutate(Source.Tissue.Type = fct_lump_n(Source.Tissue.Type, 5), 
      dx_lv = fct_recode(dcfdx_lv, "CTRL" = "1", 
                            "MCI" = "2", "MCI" = "3", 
                            "AD" = "4", "AD" = "5", "AD" = "6"), 
         dx_bl = fct_recode(dcfdx_bl, "CTRL" = "1", 
                            "MCI" = "2", "MCI" = "3", 
                            "AD" = "4", "AD" = "5", "AD" = "6")) 

mtcn_dx %>%
  ggplot(., aes(x = Source.Tissue.Type, y = mtcn_avg, colour = dx_bl)) + 
    ggbeeswarm::geom_quasirandom(dodge.width=1) + 
    facet_grid(organ ~ ., scales = "free") + 
  theme_bw()

mtcn_dx %>%
  ggplot(., aes(x = Source.Tissue.Type, y = mtcn_avg, colour = r_pd_lv)) + 
    ggbeeswarm::geom_quasirandom(dodge.width=1) + 
    facet_grid(organ ~ ., scales = "free") + 
  theme_bw()

```

### Blood cell counts and mtDNAcn
+ mtDNAcn estimates from whole blood DNA can be confounded by cell type heterogeneity, and by the presence of platelets, as platelets do not have nDNA, but have mtDNA, which artificially inflates mtDNAcn. [Han, L. et al 2019](https://dx.doi.org/10.1016/j.psyneuen.2019.04.004)
+ A subset of patients in ROSMAP have longitudinal measures of blood cell counts - can test if bimodal distribution of mtDNAcn in whole blood is due to cell type heterogeneity

```{r blood-mt}

blood_mt <- rosmap %>%
  mutate(dx_lv = fct_recode(dcfdx_lv, "CTRL" = "1", 
                            "MCI" = "2", "MCI" = "3", 
                            "AD" = "4", "AD" = "5", "AD" = "6"), 
         dx_bl = fct_recode(dcfdx_bl, "CTRL" = "1", 
                            "MCI" = "2", "MCI" = "3", 
                            "AD" = "4", "AD" = "5", "AD" = "6")) %>% 
  select(projid, study, Source.Tissue.Type,
         age_at_visit_bl, age_at_visit_lv,
         platelet_bl, rbc_bl, rdw_bl, wbc_bl,
         platelet_lv, rbc_lv, rdw_lv, wbc_lv,
         dx_bl, dx_lv,
         mtcn_avg) %>% 
  pivot_longer(c(-study, -Source.Tissue.Type, -projid, -mtcn_avg, -dx_lv, -dx_bl, -age_at_visit_bl, -age_at_visit_lv), 
               names_to = "cells", values_to = "value")
  
blood_mt %>% 
  filter(Source.Tissue.Type == "Whole Blood") %>%
  filter(str_detect(cells, "platelet_lv|rbc_lv|rdw_lv|wbc_lv")) %>%
  ggplot(., aes(x = value, y = mtcn_avg, colour = dx_lv)) + 
    facet_grid(. ~ cells, scales = "free") + 
    geom_point() + 
    theme_bw() + 
    theme(legend.position = "bottom")

blood_mt %>% 
  filter(Source.Tissue.Type == "Whole Blood") %>%
  filter(str_detect(cells, "platelet_bl|rbc_bl|rdw_bl|wbc_bl")) %>%
  ggplot(., aes(x = value, y = mtcn_avg, colour = dx_lv)) + 
    facet_grid(. ~ cells, scales = "free") + 
    geom_point() +
    theme_bw() + 
    theme(legend.position = "bottom")

```

## Clinical Diagnosis 
+ [Clinical cognitive diagnosis summary](https://www.radc.rush.edu/docs/var/detail.htm?category=Clinical+Diagnosis&subcategory=Final+consensus+diagnosis&variable=cogdx): `dcfdx` Physician’s overall cognitive diagnostic category
  * 1 =	NCI: No cognitive impairment (No impaired domains)
  * 2 =	MCI: Mild cognitive impairment (One impaired domain) and NO other cause of CI
  * 3 =	MCI: Mild cognitive impairment (One impaired domain) AND another cause of CI
  * 4 =	AD: Alzheimer’s dementia and NO other cause of CI (NINCDS PROB AD)
  * 5 =	AD: Alzheimer’s dementia AND another cause of CI (NINCDS POSS AD)
  * 6 =	Other dementia: Other primary cause of dementia
+ [Age at first Alzheimer's dementia dx](https://www.radc.rush.edu/docs/var/detail.htm?category=Clinical+Diagnosis&subcategory=Dementia&variable=age_first_ad_dx): `age_first_ad_dx` Age at cycle where first Alzheimer's dementia diagnosis was given
+ [Final consensus cognitive diagnosis](https://www.radc.rush.edu/docs/var/detail.htm?category=Clinical+Diagnosis&subcategory=Final+consensus+diagnosis&variable=cogdx): `cogdx` Clinical consensus diagnosis of cognitive status at time of death - same coding as `dcfdx`
+ [Clinical Parkinson's disease](https://www.radc.rush.edu/docs/var/detail.htm?category=Clinical+Diagnosis&subcategory=Parkinson%27s+disease&variable=r_pd): `r_pd` is made by a clinician through review of self report questions, neurological exam (when available), cognitive testing, and interview of participant. 
  * 1 =	Highly Probable
  * 2 =	Probable
  * 3 =	Possible
  * 4 =	Not Present
+ [Stroke diagnosis](https://www.radc.rush.edu/docs/var/detail.htm?category=Clinical+Diagnosis&subcategory=Stroke&variable=r_stroke): `r_stroke` s made by a clinician through review of self report questions, neurological exam (when available), cognitive testing, and interview of participant.
  * 1 =	Highly Probable
  * 2 =	Probable
  * 3 =	Possible
  * 4 =	Not Present


```{r, rosmap-dx, echo=FALSE, results='asis'}
rosmap %>% 
  select(cogdx, age_first_ad_dx, dcfdx_bl, dcfdx_lv, r_pd_bl, r_pd_lv, r_stroke_bl, r_stroke_lv) %>% 
  diagnose()  %>% 
  mutate_if(is.numeric, round, 2) %>% 
  left_join(rosmap_dic) %>% 
  select(variables, definitions, everything(), -missing_count, -unique_rate) %>% 
  knitr::kable(caption = "Data Summary")  # %>%

```


<br> 

```{r rosmap-dx-numeric, echo=FALSE, message=FALSE, tidy=FALSE}
dx_vars <- c('cogdx', 'age_first_ad_dx', 'dcfdx_bl', 'dcfdx_lv', 
             "r_pd_bl", "r_pd_lv", "r_stroke_bl", "r_stroke_lv")

dx_n <- rosmap %>% 
  select(dx_vars) %>% 
  inspect_num(.) 

dx_n %>% 
  select(-hist) %>% 
  mutate_if(is.numeric, round, 2) %>%
  knitr::kable(caption = "Variable type: Numeric") 
```

<br> 

```{r rosmap-dx-factor, echo=FALSE}
dx_c <- rosmap %>% 
  select(dx_vars) %>% 
#  mutate_if(is.factor, fct_lump_n, 4) %>%
  inspect_cat(.)   

dx_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  knitr::kable(caption = "Variable type: Factor") %>%
  kableExtra::kable_styling(full_width = F) %>%
  kableExtra::collapse_rows(columns = 1, valign = "top") 
```

### Plots 

```{r, echo=F}
dx_n %>% show_plot()

dx_c %>% show_plot(high_cardinality = 5)
```

### Cross-tabs 

```{r, echo=F}
rosmap %>% 
 gtsummary::tbl_cross(
    row = cogdx, 
    col = dcfdx_lv, 
    percent = "cell",
  ) 
```

## Pathology

[Pathology](https://www.radc.rush.edu/docs/var/overview.htm?category=Pathology): post-mortem neuropathologic evaluation

+ Alzheimer's disease 
  - [NIA-Reagan diagnosis of AD](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Alzheimer%27s+disease&variable=niareagansc): `niareagansc` modified NIA-Reagan diagnosis of Alzheimer’s disease is based on consensus recommendations for postmortem diagnosis of Alzheimer’s disease. The criteria rely on both neurofibrillary tangles (Braak) and neuritic plaques (CERAD).
    * 1	= High; 2	= Intermediate; 3	= Low; 4	= No AD
  - Dichomtomized NIA-Reagan: `ad_reagan`
  - [CERAD score](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Alzheimer%27s+disease&variable=ceradsc): `ceradsc` CERAD score is a semiquantitative measure of neuritic plaques. A CERAD neuropathologic diagnosis of AD required moderate (probable AD) or frequent neuritic plaques (definite AD) in one or more neocortical regions.
    * 1 = Definite -> frequent (C3); 2 = Probable -> moderate (C2); 3 = Possible -> Sparse (C1); 4 = No AD -> None (C0) 
  - [Braak stage](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Alzheimer%27s+disease&variable=braaksc): `braaksc` Braak Stage is a semiquantitative measure of severity of neurofibrillary tangle (NFT) pathology.
    * 0 = 0; 1 = I (entorhinal); 2 = II (entorhinal); 3 = III (limbic); 4 = IV (limbic); 5 = V (neocortical); 6 = VI (neocortical)
  - [Global AD pathology burden](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Alzheimer%27s+disease&variable=gpath): `gpath` Global AD pathology burden is a quantitative summary of AD pathology derived from counts of three AD pathologies: neuritic plaques (n), diffuse plaques (d), and neurofibrillary tangles (nft)
+ Beta-Amyloid 
  - [amyloid](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Beta-Amyloid&variable=amyloid): 	 `amyloid` Overall amyloid level - Mean of 8 brain regions
  - [plaq_d](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Beta-Amyloid&variable=plaq_d): `plaq_d` Diffuse plaque summary based on 5 regions
  - [plaq_n](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Beta-Amyloid&variable=plaq_n): `plaq_n` Neuritic plaque summary based on 5 regions
+ PHF tau Tangles 
  - [Tangle](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=PHF+tau+tangles&variable=tangles): `tangles` Tangle density - Mean of 8 brain regions 
  - [NFT burden](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=PHF+tau+tangles&variable=nft): `nft` Neurofibrillary tangle summary based on 5 regions
+ [Lewy Body disease](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Lewy+body%2fPD&variable=dlbdx): `dlbdx` Pathologic diagnosis of Lewy body diseases - 4 stages
  - 0 = Not present; 1 = nigral-predominant; 2 = limbic-type; 3 = neocortical-type
+ Vascular
  - [gross infarcts](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Vascular+-+Infarcts+(Presence+of)&variable=ci_num2_gct): `ci_num_gct` Cerebral Infarctions - Binary - Gross-Chronic-Any Location
  - [micro infarcts](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Vascular+-+Infarcts+(Presence+of)&variable=ci_num2_mct): `ci_num2_mct` Cerebral Infarctions - Binary - Micro-Chronic-Any Location
  - [Cerebral atherosclerosis](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Vascular+-+General+measures&variable=cvda_4gp2): `cvda_4gp2` Cerebral Atherosclerosis Rating 
    * 0 = None; 1 = Mild; 2 = Moderate; 3 = Severe
  - [Cerebral amyloid angiopathy](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Vascular+-+General+measures&variable=caa_4gp): `caa_4gp` Cerebral amyloid angiopathy
    * 0 = None; 1 = Mild; 2 = Moderate; 3 = Severe
  - [Arteriolosclerosis](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Vascular+-+General+measures&variable=arteriol_scler): `arteriol_scler` Arteriolosclerosis
    * 0 = None; 1 = Mild; 2 = Moderate; 3 = Severe
+ [Hippocampal sclerosis (Typical)](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=Hippocampal+sclerosis&variable=hspath_typ): `hspath_typ` Definite presence of typical hippocampal sclerosis
+ [TDP-43 stage](https://www.radc.rush.edu/docs/var/detail.htm?category=Pathology&subcategory=TDP-43&variable=tdp_st4): `tdp_st4` TDP-43 pathology from 8 regions
  - 0 = None; 1 = Amygdala; 2 = Amygdala + Limbic; 3 = Amygdala + Limbic + Neocortical
+ [Microglia](https://www.radc.rush.edu/docs/var/overview.htm?category=Pathology&subcategory=Microglia): Immunohistochemistry for microglia
  - mglia123_caud_vm 
  - mglia123_it 
  - mglia123_mf 
  - mglia123_put_p 
  - mglia23_caud_vm 
  - mglia23_it 
  - mglia23_mf 
  - mglia23_put_p 
  - mglia3_caud_vm 
  - mglia3_it 
  - mglia3_mf 
  - mglia3_put_p

```{r, rosmap-path, echo=FALSE, results='asis'}
path_vars <- c("niareagansc", "ceradsc", "braaksc", "gpath", "amyloid", "plaq_d", "plaq_n", "tangles", "nft", "dlbdx", 
         "ci_num2_gct", "ci_num2_mct", "cvda_4gp2", "caa_4gp", "arteriol_scler", "hspath_typ", "tdp_st4")

rosmap %>% 
  select(path_vars) %>% 
  diagnose()  %>% 
  mutate_if(is.numeric, round, 2) %>% 
  left_join(rosmap_dic) %>% 
  select(variables, definitions, everything(), -missing_count, -unique_rate) %>% 
  knitr::kable(caption = "Data Summary")  # %>%

```

<br> 

```{r rosmap-path-numeric, echo=FALSE, message=FALSE, tidy=FALSE}
path_n <- rosmap %>% 
  select(path_vars) %>% 
  inspect_num(.) 

path_n %>% 
  select(-hist) %>% 
  mutate_if(is.numeric, round, 2) %>%
  knitr::kable(caption = "Variable type: Numeric", table.attr = "class='dtable'") 

```

<br> 

```{r rosmap-path-factor, echo=FALSE}
path_c <- rosmap %>% 
  select(path_vars) %>% 
  mutate_if(is.factor, fct_lump_n, 10) %>%
  inspect_cat(.)   

path_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  mutate(level = replace_na(level, "NA")) %>%
  DT::datatable(caption = "Variable type: Factor", rownames = FALSE)
#  knitr::kable(caption = "Variable type: Factor") %>%
#  kableExtra::kable_styling(full_width = F)
```

### Plots 

```{r rosmap-inspect-plot, echo=F}
path_n %>% show_plot()

path_c %>% show_plot(high_cardinality = 5)

```

```{r rosmap-path-dx-plot, echo=F, warning=FALSE}
ggpubr::ggarrange(
  ## Amyloid by diagnosis 
  mutate(rosmap, niareagansc = fct_recode(niareagansc, High = "1", Intermediate = "2", Low = "3", NoAD = "4")) %>%
    ggplot(., aes(x = ceradsc, y = amyloid, color = niareagansc)) + 
      ggbeeswarm::geom_quasirandom(size = 0.75) + 
      theme_bw() + theme(legend.position = "bottom"),
    mutate(rosmap, niareagansc = fct_recode(niareagansc, High = "1", Intermediate = "2", Low = "3", NoAD = "4")) %>%
    ggplot(., aes(x = braaksc, y = nft, color = niareagansc)) + 
      ggbeeswarm::geom_quasirandom(size = 0.75) + 
      theme_bw() + theme(legend.position = "bottom"),
## Tau by diagnosis
  mutate(rosmap, dcfdx_lv = fct_recode(dcfdx_lv, NCI = "1", MCI_prob = "2", MCI_poss = "3", 
                                     AD_prob = "4", AD_poss = "5", dementia = "6")) %>%
    ggplot(., aes(x = ceradsc, y = amyloid, color = dcfdx_lv)) + 
      ggbeeswarm::geom_quasirandom(size = 0.75) + 
      theme_bw() + theme(legend.position = "bottom"), 
mutate(rosmap, dcfdx_lv = fct_recode(dcfdx_lv, NCI = "1", MCI_prob = "2", MCI_poss = "3", 
                                     AD_prob = "4", AD_poss = "5", dementia = "6")) %>%
    ggplot(., aes(x = braaksc, y = nft, color = dcfdx_lv)) + 
      ggbeeswarm::geom_quasirandom(size = 0.75) + 
      theme_bw() + theme(legend.position = "bottom")

)

```



```{r rosmap-path-crosstab-plot, echo = FALSE}
rosmap_path_crosstab <- rosmap %>% 
  select(braaksc, ceradsc, niareagansc) %>% 
  mutate(
      # braaksc = fct_recode(braaksc, B0 = "0", B1 = "1", B1 = "2", B2 = "3", B2 = "4", B3 = "5", B3 = "6"),
      # ceradsc = fct_recode(ceradsc, C0 = "4", C1 = "3", C2 = "2", C3 = "1"),
    ) %>%
  count(ceradsc, braaksc) %>% 
  left_join(
      rosmap %>% 
    select(braaksc, ceradsc, niareagansc) %>% 
    mutate(
      # braaksc = fct_recode(braaksc, B0 = "0", B1 = "1", B1 = "2", B2 = "3", B2 = "4", B3 = "5", B3 = "6"),
      # ceradsc = fct_recode(ceradsc, C0 = "4", C1 = "3", C2 = "2", C3 = "1"),
      niareagansc = fct_recode(niareagansc, High = "1", Intermediate = "2", Low = "3", NoAD = "4")
    ) %>% 
    count(ceradsc, braaksc, niareagansc) %>% 
    group_by(ceradsc, braaksc) %>% 
    arrange(-n) %>% 
    slice(1) %>% 
    ungroup() %>% 
    select(-n)
  ) %>% 
  mutate(ceradsc = fct_reorder(ceradsc, desc(ceradsc)), 
         prop = n / sum(n))

ggplot(rosmap_path_crosstab, aes(y = ceradsc, x = braaksc, label = n)) + 
  geom_tile(aes(fill = niareagansc, alpha = prop), colour = "white") + 
  geom_text() + theme_classic() +
  scale_x_discrete(position = 'top') + 
  theme_classic() + 
  scale_fill_manual(values=c("#abdda4","#fee08b","#fdae61","#f46d43","#d53e4f"),na.value = "grey90")

```


### Cross-Tabs

```{r, echo=F, warning=FALSE}
rosmap %>% 
  select(ceradsc, braaksc) %>% 
  mutate(ceradsc = as.character(ceradsc), 
         braaksc = as.character(braaksc)) %>%
 gtsummary::tbl_cross(
    row = ceradsc, 
    col = braaksc, 
    percent = "cell",
  ) 
```

<br>

```{r, echo=F}
rosmap %>% 
 gtsummary::tbl_cross(
    row = cogdx, 
    col = niareagansc, 
    percent = "row",
  ) 

```

<br>

```{r, echo = FALSE}
rosmap %>% 
  select(braaksc, ceradsc, niareagansc) %>% 
  mutate(braaksc = fct_recode(braaksc, B0 = "0", B1 = "1", B1 = "2", B2 = "3", B2 = "4", B3 = "5", B3 = "6"),
         ceradsc = fct_recode(ceradsc, C0 = "4", C1 = "3", C2 = "2", C3 = "1"),
         niareagansc = fct_recode(niareagansc, High = "1", Intermediate = "2", Low = "3", NoAD = "4")) %>% 
  count(ceradsc, braaksc, niareagansc) %>% 
  pivot_wider(names_from = niareagansc, values_from = n) %>% 
#  rename('cerad_braak' = ceradsc) %>%
#  select(-'NA') %>%
  filter(!is.na(ceradsc)) %>%
  knitr::kable(.) %>%
  kableExtra::kable_styling("striped", full_width = F) %>%
  kableExtra::collapse_rows(columns = 1, valign = "top")  


```

## Cognition 
+ [mmse](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Test+scores&variable=cts_mmse30): Mini Mental State Examination is a widely used, 30 item, standardized screening measure of dementia severity. The MMSE can be used as a surrogate measure for the CDR for the staging of dementia in AD [Perneczky et al 2006](https://doi.org/10.1097/01.JGP.0000192478.82189.a8). 

| MMSE | CDR | 
| ---- | --- |
| 30 | 0 (No) | 
| 26-29 | 0.5 (Questionable) | 
| 21-25 | 1 (Mild) | 
| 11-20 | 2 (Moderate) | 
| 0-10 | 3 (Severe) |

+ Domains 
  - [Global cognitive function](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Global+cognition&variable=cogn_global): `cogn_global`
  - [Episodic memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Domains&variable=cogn_ep): `cogn_ep`
  - [Working memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Domains&variable=cogn_wo): `cogn_wo`
  - [Semantic memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Domains&variable=cogn_se): `cogn_se`
  - [Perceptual orientation](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Domains&variable=cogn_po): `cogn_po`
  - [Perceptual speed](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Domains&variable=cogn_ps): `cogn_ps`
+ Estimated Slopes 
  - [Global cognitive function](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cogng_random_slope): `cogng_random_slope`
  - [Episodic memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cognep_random_slope): `cognep_random_slope`
  - [Working memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cognwo_random_slope): `cognwo_random_slope`
  - [Semantic memory](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cognse_random_slope): `cognse_random_slope`
  - [Perceptual orientation](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cognpo_random_slope): `cognpo_random_slope`
  - [Perceptual speed](https://www.radc.rush.edu/docs/var/detail.htm?category=Cognition&subcategory=Estimated+slopes&variable=cognps_random_slope): `cognps_random_slope`


```{r rosmap-cog, echo=FALSE}

cog_vars <- c('cts_mmse30_bl', "cogn_ep_bl", "cogn_po_bl", "cogn_ps_bl", "cogn_se_bl", "cogn_wo_bl", "cogn_global_bl", 
  'cts_mmse30_lv', "cogn_ep_lv", "cogn_po_lv", "cogn_ps_lv", "cogn_se_lv", "cogn_wo_lv", "cogn_global_lv",
  "cognep_random_slope", "cogng_random_slope", "cognpo_random_slope", "cognps_random_slope", "cognse_random_slope", "cognwo_random_slope"
  )

cog <- rosmap %>% 
  select(cog_vars) 

cog %>% 
  diagnose() 
```

```{r rosmap-cog-numeric, echo=FALSE, message=FALSE, tidy=FALSE}
cog_n <- cog %>% 
  inspect_num(.) 

cog_n %>% 
  select(-hist) %>% 
  mutate_if(is.numeric, round, 2) %>%
  knitr::kable(caption = "Variable type: Numeric", table.attr = "class='dtable'") 

```

```{r rosmap-cog-factor, echo=FALSE, eval=FALSE}
cog_c <- cog %>% 
  inspect_cat(.)   

cog_c %>% 
  pull(levels) %>% 
  bind_rows(.id = "col_name") %>% 
  rename(level = value) %>% 
  mutate_if(is.numeric, round, 2) %>% 
  mutate(level = replace_na(level, "NA")) %>%
  DT::datatable(caption = "Variable type: Factor", rownames = FALSE)
#  knitr::kable(caption = "Variable type: Factor") %>%
#  kableExtra::kable_styling(full_width = F)
```

### Plots 

```{r, echo=F}
cog_n %>% show_plot()

# cog_c %>% show_plot()
```


## Blood Measures 

+ Inflammatory markers 
  - [Age-related inflammation](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Inflammatory+markers&variable=log_agereinflam): `log_agereinflam` is a composite measure of inflammation based on levels of interleukin-1β, tumor necrosis factor-α, and interleukin-6 measured from blood samples. Only avaliable in MAP. 
  - [Ratio of good to bad inflammation](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Inflammatory+markers&variable=log_hil6_log_hil10): `log_hil6_log_hil10` is based on levels of interleukin-6 (IL-6) and interleukin-10(IL-10) measured from blood samples.
+ Routine Lab tests 
  - [Platelet (thrombocyte) count](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Routine+laboratory+tests&variable=platelet): `platelet`
  - [Red blood cell (RBC) count](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Routine+laboratory+tests&variable=rbc): `rbc`
  - [Red blood cell distribution width (RDW)](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Routine+laboratory+tests&variable=rdw): `rdw` is a measure of the range of variation of red blood cell volume. 
  - [White blood cell count](https://www.radc.rush.edu/docs/var/detail.htm?category=Blood+Measures&subcategory=Routine+laboratory+tests&variable=wbc): `wbc` 

```{r rosmap-blood, echo=FALSE}

blood_vars <- c('log_agereinflam_bl', "log_hil6_log_hil10_bl", "platelet_bl", "rbc_bl", "rdw_bl", "wbc_bl", 
                'log_agereinflam_lv', "log_hil6_log_hil10_lv", "platelet_lv", "rbc_lv", "rdw_lv", "wbc_lv")

blood <- rosmap %>% 
  select(blood_vars) 
```

```{r rosmap-blood-numeric, echo=FALSE, message=FALSE, tidy=FALSE}
blood_n <- blood %>% 
  inspect_num(.) %>% 
  mutate_if(is.numeric, round, 2) 

blood_n %>% 
  select(-hist) %>% 
   DT::datatable(caption = "Variable type: Factor", rownames = FALSE)

```

### Plots 

```{r, echo=F}
blood_n %>% show_plot()
```
