<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease</title>
  <meta name="description" content="1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  
  <meta name="twitter:description" content="1.1 Project Updates | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

<meta name="author" content="Dr. Shea Andrews" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-intro.html"/>
<link rel="next" href="1-2-neuropathological-confirmed-ad.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.15/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">mtDNAcn in AD</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html"><i class="fa fa-check"></i><b>1.1</b> Project Updates</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#to-do"><i class="fa fa-check"></i><b>1.1.1</b> To Do</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section"><i class="fa fa-check"></i><b>1.1.2</b> 2020-09-09</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-1"><i class="fa fa-check"></i><b>1.1.3</b> 2020-09-04</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-2"><i class="fa fa-check"></i><b>1.1.4</b> 2020-09-02</a></li>
<li class="chapter" data-level="1.1.5" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-3"><i class="fa fa-check"></i><b>1.1.5</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.6" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-4"><i class="fa fa-check"></i><b>1.1.6</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.7" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-5"><i class="fa fa-check"></i><b>1.1.7</b> 2020-08-07</a></li>
<li class="chapter" data-level="1.1.8" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-6"><i class="fa fa-check"></i><b>1.1.8</b> 2020-08-06</a></li>
<li class="chapter" data-level="1.1.9" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-7"><i class="fa fa-check"></i><b>1.1.9</b> 2020-08-05</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html"><i class="fa fa-check"></i><b>1.2</b> Neuropathological Confirmed AD</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#cerad-criteria---1991"><i class="fa fa-check"></i><b>1.2.1</b> CERAD Criteria - 1991</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-reagan-criteria---1997"><i class="fa fa-check"></i><b>1.2.2</b> NIA-Reagan Criteria - 1997</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-aa-criteria---2012"><i class="fa fa-check"></i><b>1.2.3</b> NIA-AA Criteria - 2012</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html"><i class="fa fa-check"></i><b>2.1</b> Mitochondrial DNA</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#adni"><i class="fa fa-check"></i><b>2.1.1</b> ADNI</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#amp-ad"><i class="fa fa-check"></i><b>2.1.2</b> AMP-AD</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#dian"><i class="fa fa-check"></i><b>2.1.3</b> DIAN</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#calling-mitochondrial-variants-in-amp-ad-dian"><i class="fa fa-check"></i><b>2.1.4</b> Calling Mitochondrial Variants in AMP-AD &amp; DIAN</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html"><i class="fa fa-check"></i><b>2.2</b> Haplogroup Assignment</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html#haplogrep"><i class="fa fa-check"></i><b>2.2.1</b> Haplogrep</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html"><i class="fa fa-check"></i><b>2.3</b> Estimating mtDNAcn</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html#mosedepth"><i class="fa fa-check"></i><b>2.3.1</b> Mosedepth</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html"><i class="fa fa-check"></i><b>2.4</b> Cell Type Enrichment</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html#xcell"><i class="fa fa-check"></i><b>2.4.1</b> xCell</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html"><i class="fa fa-check"></i><b>2.5</b> Cohorts</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>2.5.1</b> ROSMAP</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>2.5.2</b> Mayo</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>2.5.3</b> MSBB</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#amp-ad-cross-study-rnaseq-harmonization"><i class="fa fa-check"></i><b>2.5.4</b> AMP-AD Cross-Study RNAseq Harmonization</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>3</b> ROSMAP</a><ul>
<li class="chapter" data-level="3.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html"><i class="fa fa-check"></i><b>3.1</b> Demographics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>3.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-2-genetics.html"><a href="3-2-genetics.html"><i class="fa fa-check"></i><b>3.2</b> Genetics</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>3.2.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html"><i class="fa fa-check"></i><b>3.3</b> Mitochondria</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>3.3.1</b> Plots</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.2</b> diagnosis and mtDNAcn</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#blood-cell-counts-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.3</b> Blood cell counts and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html"><i class="fa fa-check"></i><b>3.4</b> RNA-seq</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html#plots-3"><i class="fa fa-check"></i><b>3.4.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html"><i class="fa fa-check"></i><b>3.5</b> Clinical Diagnosis</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#plots-4"><i class="fa fa-check"></i><b>3.5.1</b> Plots</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>3.5.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-pathology.html"><a href="3-6-pathology.html"><i class="fa fa-check"></i><b>3.6</b> Pathology</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-pathology.html"><a href="3-6-pathology.html#plots-5"><i class="fa fa-check"></i><b>3.6.1</b> Plots</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-pathology.html"><a href="3-6-pathology.html#cross-tabs-1"><i class="fa fa-check"></i><b>3.6.2</b> Cross-Tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="3-7-cognition.html"><a href="3-7-cognition.html"><i class="fa fa-check"></i><b>3.7</b> Cognition</a><ul>
<li class="chapter" data-level="3.7.1" data-path="3-7-cognition.html"><a href="3-7-cognition.html#plots-6"><i class="fa fa-check"></i><b>3.7.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html"><i class="fa fa-check"></i><b>3.8</b> Blood Measures</a><ul>
<li class="chapter" data-level="3.8.1" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html#plots-7"><i class="fa fa-check"></i><b>3.8.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.9" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html"><i class="fa fa-check"></i><b>3.9</b> Neuroimaging</a><ul>
<li class="chapter" data-level="3.9.1" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html#plots-8"><i class="fa fa-check"></i><b>3.9.1</b> Plots</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>4</b> MSBB</a><ul>
<li class="chapter" data-level="4.1" data-path="4-1-msbb-pathology.html"><a href="4-1-msbb-pathology.html"><i class="fa fa-check"></i><b>4.1</b> MSBB Pathology</a><ul>
<li class="chapter" data-level="4.1.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>4.1.1</b> Cross-tabs</a></li>
<li class="chapter" data-level="4.1.2" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>4.1.2</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-2-msbb-other-variables.html"><a href="4-2-msbb-other-variables.html"><i class="fa fa-check"></i><b>4.2</b> MSBB Other Variables</a></li>
<li class="chapter" data-level="4.3" data-path="4-3-msbb-rna-seq.html"><a href="4-3-msbb-rna-seq.html"><i class="fa fa-check"></i><b>4.3</b> MSBB RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>5</b> MAYO</a><ul>
<li class="chapter" data-level="5.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#demographics"><i class="fa fa-check"></i><b>5.1</b> Demographics</a><ul>
<li class="chapter" data-level="5.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>5.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-2-diagnosis-and-pathology.html"><a href="5-2-diagnosis-and-pathology.html"><i class="fa fa-check"></i><b>5.2</b> Diagnosis and Pathology</a><ul>
<li class="chapter" data-level="5.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>5.2.1</b> Plots</a></li>
<li class="chapter" data-level="5.2.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>5.2.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-genetics-mitochondria.html"><a href="5-3-genetics-mitochondria.html"><i class="fa fa-check"></i><b>5.3</b> Genetics &amp; Mitochondria</a><ul>
<li class="chapter" data-level="5.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>5.3.1</b> Plots</a></li>
<li class="chapter" data-level="5.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>5.3.2</b> diagnosis and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-4-rnaseq.html"><a href="5-4-rnaseq.html"><i class="fa fa-check"></i><b>5.4</b> RNAseq</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-analysis.html"><a href="6-analysis.html"><i class="fa fa-check"></i><b>6</b> Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-statistical-analysis-plan.html"><a href="6-1-statistical-analysis-plan.html"><i class="fa fa-check"></i><b>6.1</b> Statistical Analysis Plan</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html"><i class="fa fa-check"></i><b>6.2</b> Data Wrangling</a><ul>
<li class="chapter" data-level="6.2.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>6.2.1</b> ROSMAP</a></li>
<li class="chapter" data-level="6.2.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>6.2.2</b> MSBB</a></li>
<li class="chapter" data-level="6.2.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>6.2.3</b> Mayo</a></li>
<li class="chapter" data-level="6.2.4" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html#joint"><i class="fa fa-check"></i><b>6.2.4</b> Joint</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html"><i class="fa fa-check"></i><b>6.3</b> Aim 1 (mtHgs)</a><ul>
<li class="chapter" data-level="6.3.1" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1a"><i class="fa fa-check"></i><b>6.3.1</b> Aim 1a</a></li>
<li class="chapter" data-level="6.3.2" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1b"><i class="fa fa-check"></i><b>6.3.2</b> Aim 1b</a></li>
<li class="chapter" data-level="6.3.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#sensetivity-analysis"><i class="fa fa-check"></i><b>6.3.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html"><i class="fa fa-check"></i><b>6.4</b> Aim 2 (mtDNAcn)</a><ul>
<li class="chapter" data-level="6.4.1" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2a"><i class="fa fa-check"></i><b>6.4.1</b> Aim 2a</a></li>
<li class="chapter" data-level="6.4.2" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2b"><i class="fa fa-check"></i><b>6.4.2</b> Aim 2b</a></li>
<li class="chapter" data-level="6.4.3" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#sensetivity-analysis-1"><i class="fa fa-check"></i><b>6.4.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><a href="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><i class="fa fa-check"></i><b>6.5</b> Assocation of mtDNAcn with Clinical AD vs ADNC</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-aim-3.html"><a href="6-6-aim-3.html"><i class="fa fa-check"></i><b>6.6</b> Aim 3</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-aaic-abstract.html"><a href="7-aaic-abstract.html"><i class="fa fa-check"></i><b>7</b> AAIC Abstract</a></li>
<li class="chapter" data-level="8" data-path="8-final-words.html"><a href="8-final-words.html"><i class="fa fa-check"></i><b>8</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Assessing the causal association of mtDNAcn with Alzheimer’s disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="project-updates" class="section level2">
<h2><span class="header-section-number">1.1</span> Project Updates</h2>
<p>Updates to project - just like a wet lab Lab Book</p>
<div id="to-do" class="section level3">
<h3><span class="header-section-number">1.1.1</span> To Do</h3>
</div>
<div id="section" class="section level3">
<h3><span class="header-section-number">1.1.2</span> 2020-09-09</h3>
<ul>
<li>xCell: Got xCell working in R using MSBB bulk gene expression data.
<ul>
<li>Was using normalized (library size) gene expression data and adjusted for covariates. Need to use raw counts and adjust for gene length</li>
</ul></li>
</ul>
</div>
<div id="section-1" class="section level3">
<h3><span class="header-section-number">1.1.3</span> 2020-09-04</h3>
<ul>
<li>Initial analysis of haplogroup assocations with ROSMAP pathology data. Some initial positive results between haplogroup K and tau pathology.</li>
</ul>
</div>
<div id="section-2" class="section level3">
<h3><span class="header-section-number">1.1.4</span> 2020-09-02</h3>
<ul>
<li>Ordinal Logistic Regression
<ul>
<li>Looking at running OLR for neuropathology outcomes in ROSMAP in particular. Cannot (well you technicaly can) use a linear regression as the assumption of LM will be violated, in particualr the constant marginal effect where the distances between successive points in the dependent variable are assumed to be identical - clearly not the case with a ordinal variable. <a href="https://dx.doi.org/10.1016/j.jclinepi.2005.09.007">Norris, C., et al. 2006</a></li>
<li>Proportional odds assumption: relationship between each pair of outcome groups is the same, such that the coefficients that describe the relationship between, say, the lowest versus all higher categories of the response variable are the same as those that describe the relationship between the next lowest category and all higher categories, etc.
<ul>
<li>Can check assumption using a LRT or a Wald test (Yee pg 397), also graphically (Harrel; UCLA)</li>
<li>Assumption is frequently violated, however, the model can still be powerful and usefull (Harrell pg 313); the practical
implications of violating this assumption are minimal. Assumption is sensetive to other misspeicifcations.</li>
<li>Can try using a different link function or introducing interactions; or partial proportional odds</li>
</ul></li>
</ul></li>
<li>References
<ul>
<li>Abreu, M., Siqueira, A., Caiaffa, W. (2009). Regressão logística ordinal em estudos epidemiológicos. <a href="https://dx.doi.org/10.1590/s0034-89102009000100025">Revista de Saúde Pública 43(1), 183-194</a></li>
<li>Yee, T. (2015). Vector Generalized Linear and Additive Models, With an Implementation in R. <a href="https://dx.doi.org/10.1007/978-1-4939-2818-7">Springer</a></li>
<li>Harrell, F. (2015). Regression Modeling Strategies, With Applications to Linear Models, Logistic and Ordinal Regression, and Survival Analysis. <a href="https://dx.doi.org/10.1007/978-3-319-19425-7">Springer</a></li>
<li>UCLA statiscal consulting: <a href="https://stats.idre.ucla.edu/r/dae/ordinal-logistic-regression/" class="uri">https://stats.idre.ucla.edu/r/dae/ordinal-logistic-regression/</a></li>
</ul></li>
<li>R Packages
<ul>
<li>VGAM: Conducting OLR using <code>vlgm</code></li>
<li>MASS: Conducting OLR using <code>polyr</code></li>
<li>Brant: Checks proportional odds assumption when using <code>polyr</code> (also aproximate wald test)</li>
<li>QMSS: Checks proportional odds assumption when using <code>vlgm</code> (LR test)</li>
</ul></li>
<li>Sample size for logistic regression
<ul>
<li>Rule of thumb is a minimum sample size for logistic regression of ~10 events per predictor paramater <a href="https://dx.doi.org/10.1002/sim.7992">Riley, R. et al. 2019</a></li>
<li>However, this is context specific, with the appropiate sample size depending not only on the number of events relative to the number of candidate predictor parameters but also on 1) the total number of participants, 2) the outcome proportion (incidence) in the study population and; 3) the expected predictive performance of the model</li>
</ul></li>
</ul>
</div>
<div id="section-3" class="section level3">
<h3><span class="header-section-number">1.1.5</span> 2020-08-22</h3>
<ul>
<li>Initial visulizations of blood cell counts vs mtDNAcn do not show a strong relationship that would indicate a reason for the observed bimodal distribution in the samples with DNA isolated from blood.</li>
<li>Maybe due to a batch effect? May be observable if we had date of visit to see if mtDNAcn distribution changes after a given date.</li>
</ul>
</div>
<div id="section-4" class="section level3">
<h3><span class="header-section-number">1.1.6</span> 2020-08-22</h3>
<ul>
<li>Succesfully used bc.bio to call conduct joint calling of AMP-AD + DIAN samples. At a macro haplogroup level, there is little difference in haplogroup assignments. For haplogroups, ~500 samples have a different haplogroup assignment.</li>
<li>Obtained additional ROSMAP datasets including information on <a href="https://www.radc.rush.edu/docs/var/overview.htm?category=Blood+Measures&amp;subcategory=Routine+laboratory+tests">blood cell counts</a>. Plan on evaluting assocation of mtDNAcn with platlets.</li>
<li>mtDNAcn estimates from whole blood DNA can be confounded by cell type heterogeneity, and by the presence of platelets, as platelets do not have nDNA, but have mtDNA, which artificially inflates mtDNAcn. <a href="https://dx.doi.org/10.1016/j.psyneuen.2019.04.004">Han, L. et al 2019</a></li>
<li>cell counts for samples with available RNA-sequencing data can be deconvoluted using gene expression measured in whole blood. <a href="https://dx.doi.org/10.1101/2020.07.17.209023">Yang, S. et al 2020</a>
<ul>
<li>Aran, D., Hu, Z., Butte, A. (2017). xCell: digitally portraying the tissue cellular heterogeneity landscape. <a href="https://dx.doi.org/10.1186/s13059-017-1349-1">Genome Biology 18(1), 220</a></li>
</ul></li>
</ul>
</div>
<div id="section-5" class="section level3">
<h3><span class="header-section-number">1.1.7</span> 2020-08-07</h3>
<ul>
<li>trying to bcbio working
<ul>
<li>got it working on test set of five samples</li>
<li>used sed to rename chr lables in DIAN</li>
</ul></li>
<li>Not working on full sample list…</li>
</ul>
</div>
<div id="section-6" class="section level3">
<h3><span class="header-section-number">1.1.8</span> 2020-08-06</h3>
<ul>
<li>SampleIDs for DIAN
<ul>
<li>changed from WGS ID (file name) to sample ID</li>
<li>causes issues with fastmitocalc, which uses file name as prefix in output</li>
</ul></li>
<li>bcbio.variation.recall
<ul>
<li>Does not work when samples are mapped to differen builds. while MT positions are the same, chrm label is different which means the samples on other builds are not recalled.</li>
</ul></li>
<li>installed <a href="https://crossmap.readthedocs.io/en/latest/">crossmap</a> using conda install rather than pip
<ul>
<li>crossmap does not work due to chain files using chrM as label, does not lift over correctly</li>
<li>solution - use sed to rename all instances of chrM (or MT) to MT. Worked on example file from DIAN. allowed bcbio to work</li>
</ul></li>
</ul>
</div>
<div id="section-7" class="section level3">
<h3><span class="header-section-number">1.1.9</span> 2020-08-05</h3>
<ul>
<li>Looking at conducting joint calling using bcbio
<ul>
<li>Issues with DIAN files, in particular it uses hg38, while AMP-AD uses 37
<ul>
<li>Not sure if this will be an issue, may just be able to use the same reference fasta</li>
<li>Tried using <a href="https://crossmap.readthedocs.io/en/latest/">crossmap</a> to liftover bam files, however mtDNA was unmapped, renames mtDNA as “M” rathern than “MT”, also was rather dodgy to get working on minerva, ended up testing it locally</li>
</ul></li>
<li>DIAN sampleIDs are different from file names, which must match read groups in bam files for bcbio</li>
</ul></li>
<li>Redoing sampleIDs for DIAN
<ul>
<li>encounted issue with broken symlinks - took a while to diagnoses and fix</li>
<li>pipeline is working again</li>
</ul></li>
</ul>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="1-2-neuropathological-confirmed-ad.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["notebook.pdf", "notebook.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
