<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease</title>
  <meta name="description" content="Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  
  <meta name="twitter:description" content="Chapter 3 ROSMAP | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

<meta name="author" content="Dr. Shea Andrews" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-5-cohorts.html"/>
<link rel="next" href="3-1-demographics.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.15/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">mtDNAcn in AD</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html"><i class="fa fa-check"></i><b>1.1</b> Project Updates</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#to-do"><i class="fa fa-check"></i><b>1.1.1</b> To Do</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section"><i class="fa fa-check"></i><b>1.1.2</b> 2020-09-09</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-1"><i class="fa fa-check"></i><b>1.1.3</b> 2020-09-04</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-2"><i class="fa fa-check"></i><b>1.1.4</b> 2020-09-02</a></li>
<li class="chapter" data-level="1.1.5" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-3"><i class="fa fa-check"></i><b>1.1.5</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.6" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-4"><i class="fa fa-check"></i><b>1.1.6</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.7" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-5"><i class="fa fa-check"></i><b>1.1.7</b> 2020-08-07</a></li>
<li class="chapter" data-level="1.1.8" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-6"><i class="fa fa-check"></i><b>1.1.8</b> 2020-08-06</a></li>
<li class="chapter" data-level="1.1.9" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-7"><i class="fa fa-check"></i><b>1.1.9</b> 2020-08-05</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html"><i class="fa fa-check"></i><b>1.2</b> Neuropathological Confirmed AD</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#cerad-criteria---1991"><i class="fa fa-check"></i><b>1.2.1</b> CERAD Criteria - 1991</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-reagan-criteria---1997"><i class="fa fa-check"></i><b>1.2.2</b> NIA-Reagan Criteria - 1997</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-aa-criteria---2012"><i class="fa fa-check"></i><b>1.2.3</b> NIA-AA Criteria - 2012</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html"><i class="fa fa-check"></i><b>2.1</b> Mitochondrial DNA</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#adni"><i class="fa fa-check"></i><b>2.1.1</b> ADNI</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#amp-ad"><i class="fa fa-check"></i><b>2.1.2</b> AMP-AD</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#dian"><i class="fa fa-check"></i><b>2.1.3</b> DIAN</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#calling-mitochondrial-variants-in-amp-ad-dian"><i class="fa fa-check"></i><b>2.1.4</b> Calling Mitochondrial Variants in AMP-AD &amp; DIAN</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html"><i class="fa fa-check"></i><b>2.2</b> Haplogroup Assignment</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html#haplogrep"><i class="fa fa-check"></i><b>2.2.1</b> Haplogrep</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html"><i class="fa fa-check"></i><b>2.3</b> Estimating mtDNAcn</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html#mosedepth"><i class="fa fa-check"></i><b>2.3.1</b> Mosedepth</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html"><i class="fa fa-check"></i><b>2.4</b> Cell Type Enrichment</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html#xcell"><i class="fa fa-check"></i><b>2.4.1</b> xCell</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html"><i class="fa fa-check"></i><b>2.5</b> Cohorts</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>2.5.1</b> ROSMAP</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>2.5.2</b> Mayo</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>2.5.3</b> MSBB</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#amp-ad-cross-study-rnaseq-harmonization"><i class="fa fa-check"></i><b>2.5.4</b> AMP-AD Cross-Study RNAseq Harmonization</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>3</b> ROSMAP</a><ul>
<li class="chapter" data-level="3.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html"><i class="fa fa-check"></i><b>3.1</b> Demographics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>3.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-2-genetics.html"><a href="3-2-genetics.html"><i class="fa fa-check"></i><b>3.2</b> Genetics</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>3.2.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html"><i class="fa fa-check"></i><b>3.3</b> Mitochondria</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>3.3.1</b> Plots</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.2</b> diagnosis and mtDNAcn</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#blood-cell-counts-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.3</b> Blood cell counts and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html"><i class="fa fa-check"></i><b>3.4</b> RNA-seq</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html#plots-3"><i class="fa fa-check"></i><b>3.4.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html"><i class="fa fa-check"></i><b>3.5</b> Clinical Diagnosis</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#plots-4"><i class="fa fa-check"></i><b>3.5.1</b> Plots</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>3.5.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-pathology.html"><a href="3-6-pathology.html"><i class="fa fa-check"></i><b>3.6</b> Pathology</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-pathology.html"><a href="3-6-pathology.html#plots-5"><i class="fa fa-check"></i><b>3.6.1</b> Plots</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-pathology.html"><a href="3-6-pathology.html#cross-tabs-1"><i class="fa fa-check"></i><b>3.6.2</b> Cross-Tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="3-7-cognition.html"><a href="3-7-cognition.html"><i class="fa fa-check"></i><b>3.7</b> Cognition</a><ul>
<li class="chapter" data-level="3.7.1" data-path="3-7-cognition.html"><a href="3-7-cognition.html#plots-6"><i class="fa fa-check"></i><b>3.7.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html"><i class="fa fa-check"></i><b>3.8</b> Blood Measures</a><ul>
<li class="chapter" data-level="3.8.1" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html#plots-7"><i class="fa fa-check"></i><b>3.8.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.9" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html"><i class="fa fa-check"></i><b>3.9</b> Neuroimaging</a><ul>
<li class="chapter" data-level="3.9.1" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html#plots-8"><i class="fa fa-check"></i><b>3.9.1</b> Plots</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>4</b> MSBB</a><ul>
<li class="chapter" data-level="4.1" data-path="4-1-msbb-pathology.html"><a href="4-1-msbb-pathology.html"><i class="fa fa-check"></i><b>4.1</b> MSBB Pathology</a><ul>
<li class="chapter" data-level="4.1.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>4.1.1</b> Cross-tabs</a></li>
<li class="chapter" data-level="4.1.2" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>4.1.2</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-2-msbb-other-variables.html"><a href="4-2-msbb-other-variables.html"><i class="fa fa-check"></i><b>4.2</b> MSBB Other Variables</a></li>
<li class="chapter" data-level="4.3" data-path="4-3-msbb-rna-seq.html"><a href="4-3-msbb-rna-seq.html"><i class="fa fa-check"></i><b>4.3</b> MSBB RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>5</b> MAYO</a><ul>
<li class="chapter" data-level="5.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#demographics"><i class="fa fa-check"></i><b>5.1</b> Demographics</a><ul>
<li class="chapter" data-level="5.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>5.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-2-diagnosis-and-pathology.html"><a href="5-2-diagnosis-and-pathology.html"><i class="fa fa-check"></i><b>5.2</b> Diagnosis and Pathology</a><ul>
<li class="chapter" data-level="5.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>5.2.1</b> Plots</a></li>
<li class="chapter" data-level="5.2.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>5.2.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-genetics-mitochondria.html"><a href="5-3-genetics-mitochondria.html"><i class="fa fa-check"></i><b>5.3</b> Genetics &amp; Mitochondria</a><ul>
<li class="chapter" data-level="5.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>5.3.1</b> Plots</a></li>
<li class="chapter" data-level="5.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>5.3.2</b> diagnosis and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-4-rnaseq.html"><a href="5-4-rnaseq.html"><i class="fa fa-check"></i><b>5.4</b> RNAseq</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-analysis.html"><a href="6-analysis.html"><i class="fa fa-check"></i><b>6</b> Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-statistical-analysis-plan.html"><a href="6-1-statistical-analysis-plan.html"><i class="fa fa-check"></i><b>6.1</b> Statistical Analysis Plan</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html"><i class="fa fa-check"></i><b>6.2</b> Data Wrangling</a><ul>
<li class="chapter" data-level="6.2.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>6.2.1</b> ROSMAP</a></li>
<li class="chapter" data-level="6.2.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>6.2.2</b> MSBB</a></li>
<li class="chapter" data-level="6.2.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>6.2.3</b> Mayo</a></li>
<li class="chapter" data-level="6.2.4" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html#joint"><i class="fa fa-check"></i><b>6.2.4</b> Joint</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html"><i class="fa fa-check"></i><b>6.3</b> Aim 1 (mtHgs)</a><ul>
<li class="chapter" data-level="6.3.1" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1a"><i class="fa fa-check"></i><b>6.3.1</b> Aim 1a</a></li>
<li class="chapter" data-level="6.3.2" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1b"><i class="fa fa-check"></i><b>6.3.2</b> Aim 1b</a></li>
<li class="chapter" data-level="6.3.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#sensetivity-analysis"><i class="fa fa-check"></i><b>6.3.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html"><i class="fa fa-check"></i><b>6.4</b> Aim 2 (mtDNAcn)</a><ul>
<li class="chapter" data-level="6.4.1" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2a"><i class="fa fa-check"></i><b>6.4.1</b> Aim 2a</a></li>
<li class="chapter" data-level="6.4.2" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2b"><i class="fa fa-check"></i><b>6.4.2</b> Aim 2b</a></li>
<li class="chapter" data-level="6.4.3" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#sensetivity-analysis-1"><i class="fa fa-check"></i><b>6.4.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><a href="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><i class="fa fa-check"></i><b>6.5</b> Assocation of mtDNAcn with Clinical AD vs ADNC</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-aim-3.html"><a href="6-6-aim-3.html"><i class="fa fa-check"></i><b>6.6</b> Aim 3</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-aaic-abstract.html"><a href="7-aaic-abstract.html"><i class="fa fa-check"></i><b>7</b> AAIC Abstract</a></li>
<li class="chapter" data-level="8" data-path="8-final-words.html"><a href="8-final-words.html"><i class="fa fa-check"></i><b>8</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Assessing the causal association of mtDNAcn with Alzheimer’s disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="rosmap" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> ROSMAP</h1>
<p>The samples that we have profiled come from two prospective studies of aging-The Religious order Study (ROS) and the Memory and Aging Project (MAP)-that recruit older individuals without known dementia and include (1) detailed cognitive, neuroimaging and other ante-mortem phenotyping and (2) an autopsy at the time of death that includes a structured neuropathologic examination. A subset of the ROSMAP samples (n=1200 for 1179 unique deceased participants) underwent whole genome sequencing, with DNA coming from brain tissue (n=806), whole blood (n=389) or lymphocytes transformed with EBV virus (n=5) (<span class="citation">Jager et al. (<a href="#ref-10.1038/sdata.2018.142">2018</a>)</span>).</p>
<p>Data Dictionaries for ROSMAP can be found at:</p>
<ul>
<li><a href="https://adknowledgeportal.synapse.org/Explore/Studies?Study=syn3219045">AMP-AD</a></li>
<li><a href="https://www.radc.rush.edu/docs/var/variables.htm">RADC</a></li>
</ul>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="co">## Phenotypic data</span></a>
<a class="sourceLine" id="cb1-2" data-line-number="2">rosmap.pheno_bl &lt;-<span class="st"> </span>readxl<span class="op">::</span><span class="kw">read_xlsx</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/dataset_899_basic_08-22-2020.xlsx&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb1-3" data-line-number="3"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">projid =</span> <span class="kw">as.numeric</span>(projid))</a>
<a class="sourceLine" id="cb1-4" data-line-number="4">rosmap.pheno_long.raw &lt;-<span class="st"> </span>readxl<span class="op">::</span><span class="kw">read_xlsx</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/dataset_899_long_08-22-2020.xlsx&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb1-5" data-line-number="5"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">projid =</span> <span class="kw">as.numeric</span>(projid))</a>
<a class="sourceLine" id="cb1-6" data-line-number="6"></a>
<a class="sourceLine" id="cb1-7" data-line-number="7"><span class="co">## Activities of daily living</span></a>
<a class="sourceLine" id="cb1-8" data-line-number="8">rosmap.adl.raw &lt;-<span class="st"> </span>readxl<span class="op">::</span><span class="kw">read_xlsx</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/dataset_641_long_04-29-2020.xlsx&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb1-9" data-line-number="9"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">projid =</span> <span class="kw">as.numeric</span>(projid)) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-10" data-line-number="10"><span class="st">  </span><span class="kw">select</span>(projid, study, fu_year, iadlsum, katzsum)</a>
<a class="sourceLine" id="cb1-11" data-line-number="11"></a>
<a class="sourceLine" id="cb1-12" data-line-number="12"><span class="co"># wgs_clinical.raw &lt;- read_csv(&#39;data/AMPAD_extra/rosmap/ROSMAP_Clinical_2019-05_v3.csv&#39;)</span></a>
<a class="sourceLine" id="cb1-13" data-line-number="13"><span class="co">## Mitochondrial </span></a>
<a class="sourceLine" id="cb1-14" data-line-number="14">rosmap.wgsqc &lt;-<span class="st"> </span><span class="kw">read_csv</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/WGS_sample_QC_info.csv&quot;</span>, <span class="dt">guess_max =</span> <span class="dv">10000</span>)</a>
<a class="sourceLine" id="cb1-15" data-line-number="15">mosdepth &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&quot;data/mosdepth/mosdepth_mtDNAcn_All.txt&quot;</span>)</a>
<a class="sourceLine" id="cb1-16" data-line-number="16">haplogrep &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&quot;data/haplogrep/haplogrep_jointAll.txt&quot;</span>)</a>
<a class="sourceLine" id="cb1-17" data-line-number="17"></a>
<a class="sourceLine" id="cb1-18" data-line-number="18">apoe &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/wgs_apoe.tsv&quot;</span>)</a>
<a class="sourceLine" id="cb1-19" data-line-number="19"></a>
<a class="sourceLine" id="cb1-20" data-line-number="20">xcell &lt;-<span class="st"> </span><span class="kw">read_csv</span>(<span class="st">&quot;data/xcell/ampad_xCell.csv&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-21" data-line-number="21"><span class="st">  </span><span class="kw">filter</span>(study <span class="op">==</span><span class="st"> &quot;ROSMAP&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-22" data-line-number="22"><span class="st">  </span><span class="kw">select</span>(<span class="op">-</span>SampleID, <span class="op">-</span>study) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-23" data-line-number="23"><span class="st">  </span><span class="kw">rename</span>(<span class="dt">rna_seq_tissue =</span> Tissue, <span class="dt">rna_seq_batch =</span> batch) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb1-24" data-line-number="24"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">ID =</span> <span class="kw">as.numeric</span>(ID))</a>
<a class="sourceLine" id="cb1-25" data-line-number="25"></a>
<a class="sourceLine" id="cb1-26" data-line-number="26"><span class="co"># Neuroimaging </span></a>
<a class="sourceLine" id="cb1-27" data-line-number="27">tot_vol.raw &lt;-<span class="st"> </span><span class="kw">bind_rows</span>(</a>
<a class="sourceLine" id="cb1-28" data-line-number="28">  <span class="kw">read_csv</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/mri_total_volumes_share_ROS.csv&quot;</span>), </a>
<a class="sourceLine" id="cb1-29" data-line-number="29">  <span class="kw">read_csv</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/mri_total_volumes_share_MAP.csv&quot;</span>)</a>
<a class="sourceLine" id="cb1-30" data-line-number="30">) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-31" data-line-number="31"><span class="st">  </span><span class="kw">select</span>(<span class="op">-</span>X9) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-32" data-line-number="32"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">projid =</span> <span class="kw">as.numeric</span>(projid), </a>
<a class="sourceLine" id="cb1-33" data-line-number="33">         <span class="dt">visit =</span> <span class="kw">as.numeric</span>(visit))</a>
<a class="sourceLine" id="cb1-34" data-line-number="34">subcort_v6 &lt;-<span class="st"> </span><span class="kw">read_csv</span>(<span class="st">&quot;data/AMPAD_extra/rosmap/mri_subcortical_v6_share.csv&quot;</span>)  <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb1-35" data-line-number="35"><span class="st">  </span><span class="kw">select</span>(<span class="op">-</span>X69)</a></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="co">## extract baseline and last vist values and spread</span></a>
<a class="sourceLine" id="cb2-2" data-line-number="2">rosmap.pheno_long &lt;-<span class="st"> </span>rosmap.pheno_long.raw <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="st">  </span><span class="kw">left_join</span>(rosmap.adl.raw) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-4" data-line-number="4"><span class="st">  </span><span class="kw">group_by</span>(projid) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-5" data-line-number="5"><span class="st">  </span><span class="kw">slice</span>(<span class="kw">c</span>(<span class="kw">which.min</span>(fu_year), <span class="kw">which.max</span>(fu_year))) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-6" data-line-number="6"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">visit =</span> <span class="kw">case_when</span>(fu_year <span class="op">==</span><span class="st"> </span><span class="kw">min</span>(fu_year) <span class="op">~</span><span class="st"> &quot;bl&quot;</span>, </a>
<a class="sourceLine" id="cb2-7" data-line-number="7">                           fu_year <span class="op">==</span><span class="st"> </span><span class="kw">max</span>(fu_year) <span class="op">~</span><span class="st"> &quot;lv&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-8" data-line-number="8"><span class="st">  </span><span class="kw">distinct</span>(., fu_year, <span class="dt">.keep_all =</span> <span class="ot">TRUE</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-9" data-line-number="9"><span class="st">  </span><span class="kw">ungroup</span>() <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-10" data-line-number="10"><span class="st">  </span><span class="kw">pivot_wider</span>(<span class="dt">names_from =</span> visit, <span class="dt">values_from =</span> <span class="kw">c</span>(<span class="op">-</span>projid, <span class="op">-</span>study, <span class="op">-</span>scaled_to))</a>
<a class="sourceLine" id="cb2-11" data-line-number="11"></a>
<a class="sourceLine" id="cb2-12" data-line-number="12"><span class="co">## Merge WGS files </span></a>
<a class="sourceLine" id="cb2-13" data-line-number="13">rosmap_wgs &lt;-<span class="st"> </span>rosmap.wgsqc <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-14" data-line-number="14"><span class="st">  </span><span class="kw">filter</span>(QC <span class="op">==</span><span class="st"> &quot;Pass&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-15" data-line-number="15"><span class="st">  </span><span class="kw">select</span>(projid, <span class="dt">SampleID =</span> WGS_id, Source.Tissue.Type, QC) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-16" data-line-number="16"><span class="st">  </span><span class="kw">left_join</span>(mosdepth, <span class="dt">by =</span> <span class="st">&quot;SampleID&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-17" data-line-number="17"><span class="st">  </span><span class="kw">left_join</span>(haplogrep, <span class="dt">by =</span> <span class="st">&quot;SampleID&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-18" data-line-number="18"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">macro =</span> <span class="kw">case_when</span>(</a>
<a class="sourceLine" id="cb2-19" data-line-number="19">    <span class="kw">str_detect</span>(Haplogroup, <span class="st">&quot;^L|^HV|^JT&quot;</span>) <span class="op">~</span><span class="st"> </span><span class="kw">substr</span>(Haplogroup, <span class="dt">start =</span> <span class="dv">1</span>, <span class="dt">stop =</span> <span class="dv">2</span>), </a>
<a class="sourceLine" id="cb2-20" data-line-number="20">    <span class="ot">TRUE</span> <span class="op">~</span><span class="st"> </span><span class="kw">substr</span>(Haplogroup, <span class="dt">start =</span> <span class="dv">1</span>, <span class="dt">stop =</span> <span class="dv">1</span>)</a>
<a class="sourceLine" id="cb2-21" data-line-number="21">  ))</a>
<a class="sourceLine" id="cb2-22" data-line-number="22"></a>
<a class="sourceLine" id="cb2-23" data-line-number="23"><span class="co">## extract last MRI measurment</span></a>
<a class="sourceLine" id="cb2-24" data-line-number="24">tot_vol &lt;-<span class="st"> </span>tot_vol.raw <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-25" data-line-number="25"><span class="st">  </span><span class="kw">group_by</span>(projid) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-26" data-line-number="26"><span class="st">  </span><span class="kw">arrange</span>(visit) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-27" data-line-number="27"><span class="st">  </span><span class="kw">slice</span>(<span class="kw">which.max</span>(visit)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-28" data-line-number="28"><span class="st">  </span><span class="kw">ungroup</span>()  </a>
<a class="sourceLine" id="cb2-29" data-line-number="29"></a>
<a class="sourceLine" id="cb2-30" data-line-number="30"><span class="co">### 32 duplicate IDS </span></a>
<a class="sourceLine" id="cb2-31" data-line-number="31"><span class="co"># get_dupes(rosmap_wgs, projid)  %&gt;% </span></a>
<a class="sourceLine" id="cb2-32" data-line-number="32"><span class="co">#   print(n = Inf)</span></a>
<a class="sourceLine" id="cb2-33" data-line-number="33"></a>
<a class="sourceLine" id="cb2-34" data-line-number="34">rosmap &lt;-<span class="st"> </span>rosmap.pheno_bl <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-35" data-line-number="35"><span class="st">  </span><span class="kw">left_join</span>(apoe, <span class="dt">by =</span> <span class="st">&quot;projid&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-36" data-line-number="36"><span class="st">  </span><span class="kw">left_join</span>(rosmap.pheno_long, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&quot;projid&quot;</span>, <span class="st">&quot;study&quot;</span>, <span class="st">&quot;scaled_to&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-37" data-line-number="37"><span class="st">  </span><span class="kw">left_join</span>(tot_vol, <span class="dt">by =</span> <span class="st">&quot;projid&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-38" data-line-number="38"><span class="st">  </span><span class="kw">left_join</span>(xcell, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&quot;projid&quot;</span> =<span class="st"> &quot;ID&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-39" data-line-number="39"><span class="st">  </span><span class="kw">left_join</span>(rosmap_wgs, <span class="dt">by =</span> <span class="st">&quot;projid&quot;</span>) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-40" data-line-number="40"><span class="st">    </span><span class="kw">mutate</span>(<span class="dt">race7 =</span> <span class="kw">as_factor</span>(race7),</a>
<a class="sourceLine" id="cb2-41" data-line-number="41">        <span class="dt">race7 =</span> <span class="kw">fct_recode</span>(race7,<span class="st">&#39;White&#39;</span> =<span class="st"> &#39;1&#39;</span>, <span class="st">&#39;Black&#39;</span> =<span class="st"> &#39;2&#39;</span>, <span class="st">&quot;AMR&quot;</span> =<span class="st"> &quot;3&quot;</span>, <span class="st">&quot;Oceania&quot;</span> =<span class="st"> &quot;4&quot;</span>, </a>
<a class="sourceLine" id="cb2-42" data-line-number="42">                               <span class="st">&quot;Asian&quot;</span> =<span class="st"> &quot;5&quot;</span>, <span class="st">&quot;Other&quot;</span> =<span class="st"> &quot;6&quot;</span>, <span class="st">&quot;Other&quot;</span> =<span class="st"> &quot;7&quot;</span>),</a>
<a class="sourceLine" id="cb2-43" data-line-number="43">        <span class="dt">z_mtdnacn =</span> <span class="kw">scale</span>(mtcn_avg, <span class="dt">center =</span> <span class="ot">TRUE</span>, <span class="dt">scale =</span> <span class="ot">TRUE</span>)[,<span class="dv">1</span>],</a>
<a class="sourceLine" id="cb2-44" data-line-number="44">        <span class="dt">spanish =</span> <span class="kw">as_factor</span>(spanish),</a>
<a class="sourceLine" id="cb2-45" data-line-number="45">        <span class="dt">spanish =</span> <span class="kw">fct_recode</span>(spanish,<span class="st">&#39;Yes&#39;</span> =<span class="st"> &#39;1&#39;</span>, <span class="st">&#39;No&#39;</span> =<span class="st"> &#39;2&#39;</span>),</a>
<a class="sourceLine" id="cb2-46" data-line-number="46">        <span class="dt">organ =</span> <span class="kw">recode</span>(Source.Tissue.Type, <span class="st">&#39;Blood&#39;</span> =<span class="st"> &#39;blood&#39;</span>, <span class="st">&#39;Blood-PBMC&#39;</span> =<span class="st"> &#39;blood&#39;</span>, <span class="st">&#39;Whole Blood&#39;</span> =<span class="st"> &#39;blood&#39;</span>, </a>
<a class="sourceLine" id="cb2-47" data-line-number="47">                       <span class="st">&#39;Blood-Cerebellum&#39;</span> =<span class="st"> &#39;brain&#39;</span>,  <span class="st">&#39;Brain-Anterior Caudate&#39;</span> =<span class="st"> &#39;brain&#39;</span>,</a>
<a class="sourceLine" id="cb2-48" data-line-number="48">                       <span class="st">&#39;Brain-Cerebellum&#39;</span> =<span class="st"> &#39;brain&#39;</span>, <span class="st">&#39;Brain-DLPFC&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-49" data-line-number="49">                       <span class="st">&#39;Brain-Frontal Cortex (BA unknown)&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-50" data-line-number="50">                       <span class="st">&#39;Brain-Frontal Pole (BA10-12,32)&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-51" data-line-number="51">                       <span class="st">&#39;Brain-Occipital Association Cortex (BA18,19)&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-52" data-line-number="52">                       <span class="st">&#39;Brain-PCC&#39;</span> =<span class="st"> &#39;brain&#39;</span>, <span class="st">&#39;Brain-Posterior Cingulate Cortex&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-53" data-line-number="53">                       <span class="st">&#39;Brain-region unknown&#39;</span> =<span class="st"> &#39;brain&#39;</span>, </a>
<a class="sourceLine" id="cb2-54" data-line-number="54">                       <span class="st">&#39;lymphocytes _transformed _with EBV virus&#39;</span>=<span class="st"> &#39;lymphocytes&#39;</span>),</a>
<a class="sourceLine" id="cb2-55" data-line-number="55">        <span class="dt">organ =</span> <span class="kw">as_factor</span>(organ),</a>
<a class="sourceLine" id="cb2-56" data-line-number="56">        <span class="dt">apoe_genotype =</span> <span class="kw">ifelse</span>(<span class="kw">is.na</span>(apoe_genotype), apoe, apoe_genotype),</a>
<a class="sourceLine" id="cb2-57" data-line-number="57">        <span class="dt">apoe4 =</span> <span class="kw">recode</span>(apoe_genotype, <span class="st">&#39;22&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;23&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;33&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;24&#39;</span> =<span class="st"> &#39;e4+&#39;</span>, <span class="st">&#39;34&#39;</span> =<span class="st"> &#39;e4+&#39;</span>, <span class="st">&#39;44&#39;</span> =<span class="st"> &#39;e4+&#39;</span>),</a>
<a class="sourceLine" id="cb2-58" data-line-number="58">        <span class="dt">tomm40_hap =</span> <span class="kw">as_factor</span>(tomm40_hap),</a>
<a class="sourceLine" id="cb2-59" data-line-number="59">        <span class="dt">tomm40_hap =</span> <span class="kw">fct_recode</span>(tomm40_hap, <span class="st">&#39;S/S&#39;</span> =<span class="st"> &#39;1&#39;</span>, <span class="st">&#39;S/L&#39;</span> =<span class="st"> &#39;2&#39;</span>, <span class="st">&quot;S/VL&quot;</span> =<span class="st"> &quot;3&quot;</span>, <span class="st">&quot;L/L&quot;</span> =<span class="st"> &quot;4&quot;</span>, <span class="st">&quot;L/VL&quot;</span> =<span class="st"> &quot;5&quot;</span>, <span class="st">&quot;VL/VL&quot;</span> =<span class="st"> &quot;6&quot;</span>),</a>
<a class="sourceLine" id="cb2-60" data-line-number="60">        <span class="dt">aod_cat =</span> <span class="kw">cut</span>(age_death, <span class="kw">c</span>(<span class="dv">50</span>, <span class="dv">60</span>, <span class="dv">70</span>, <span class="dv">80</span>, <span class="dv">90</span>, <span class="ot">Inf</span>), <span class="kw">c</span>(<span class="st">&#39;50-59&#39;</span>, <span class="st">&#39;60-69&#39;</span>, <span class="st">&#39;70-79&#39;</span>, <span class="st">&#39;80-89&#39;</span>, <span class="st">&#39;90+&#39;</span>), <span class="dt">right =</span> <span class="ot">FALSE</span>),</a>
<a class="sourceLine" id="cb2-61" data-line-number="61">        <span class="dt">aod_cat =</span> <span class="kw">ordered</span>(aod_cat, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;50-59&#39;</span>, <span class="st">&#39;60-69&#39;</span>, <span class="st">&#39;70-79&#39;</span>, <span class="st">&#39;80-89&#39;</span>, <span class="st">&#39;90+&#39;</span>)), </a>
<a class="sourceLine" id="cb2-62" data-line-number="62">        <span class="dt">msex =</span> <span class="kw">as.factor</span>(msex), </a>
<a class="sourceLine" id="cb2-63" data-line-number="63">        <span class="dt">msex =</span> <span class="kw">fct_recode</span>(msex, <span class="st">&#39;M&#39;</span> =<span class="st"> &#39;1&#39;</span>, <span class="st">&#39;F&#39;</span> =<span class="st"> &#39;0&#39;</span>),</a>
<a class="sourceLine" id="cb2-64" data-line-number="64">        <span class="dt">cogdx =</span> <span class="kw">factor</span>(cogdx), </a>
<a class="sourceLine" id="cb2-65" data-line-number="65">        <span class="dt">r_pd_bl =</span> <span class="kw">factor</span>(r_pd_bl), </a>
<a class="sourceLine" id="cb2-66" data-line-number="66">        <span class="dt">r_pd_lv =</span> <span class="kw">factor</span>(r_pd_lv), </a>
<a class="sourceLine" id="cb2-67" data-line-number="67">        <span class="dt">r_stroke_bl =</span> <span class="kw">factor</span>(r_stroke_bl), </a>
<a class="sourceLine" id="cb2-68" data-line-number="68">        <span class="dt">r_stroke_lv =</span> <span class="kw">factor</span>(r_stroke_lv), </a>
<a class="sourceLine" id="cb2-69" data-line-number="69">        <span class="dt">dcfdx_bl =</span> <span class="kw">factor</span>(dcfdx_bl),</a>
<a class="sourceLine" id="cb2-70" data-line-number="70">        <span class="dt">dcfdx_lv =</span> <span class="kw">factor</span>(dcfdx_lv),</a>
<a class="sourceLine" id="cb2-71" data-line-number="71">        <span class="dt">apoe_genotype =</span> <span class="kw">as.factor</span>(apoe_genotype), </a>
<a class="sourceLine" id="cb2-72" data-line-number="72">        <span class="dt">apoe4 =</span> <span class="kw">as.factor</span>(apoe4), </a>
<a class="sourceLine" id="cb2-73" data-line-number="73">        <span class="dt">study =</span> <span class="kw">as.factor</span>(study), </a>
<a class="sourceLine" id="cb2-74" data-line-number="74">        <span class="dt">braaksc =</span> <span class="kw">ordered</span>(braaksc, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;0&#39;</span>, <span class="st">&#39;1&#39;</span>, <span class="st">&#39;2&#39;</span>, <span class="st">&#39;3&#39;</span>, <span class="st">&#39;4&#39;</span>, <span class="st">&#39;5&#39;</span>, <span class="st">&#39;6&#39;</span>)), </a>
<a class="sourceLine" id="cb2-75" data-line-number="75">        <span class="dt">ceradsc =</span> <span class="kw">ordered</span>(ceradsc, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;4&#39;</span>, <span class="st">&#39;3&#39;</span>, <span class="st">&#39;2&#39;</span>, <span class="st">&#39;1&#39;</span>)), </a>
<a class="sourceLine" id="cb2-76" data-line-number="76">        <span class="dt">dlbdx =</span> <span class="kw">as.factor</span>(dlbdx), </a>
<a class="sourceLine" id="cb2-77" data-line-number="77">        <span class="dt">ci_num2_mct =</span> <span class="kw">as.factor</span>(ci_num2_mct), </a>
<a class="sourceLine" id="cb2-78" data-line-number="78">        <span class="dt">ci_num2_gct =</span> <span class="kw">as.factor</span>(ci_num2_gct), </a>
<a class="sourceLine" id="cb2-79" data-line-number="79">        <span class="dt">cvda_4gp2 =</span> <span class="kw">as.factor</span>(cvda_4gp2),</a>
<a class="sourceLine" id="cb2-80" data-line-number="80">        <span class="dt">caa_4gp =</span> <span class="kw">as.factor</span>(caa_4gp),</a>
<a class="sourceLine" id="cb2-81" data-line-number="81">        <span class="dt">arteriol_scler =</span> <span class="kw">as.factor</span>(arteriol_scler),</a>
<a class="sourceLine" id="cb2-82" data-line-number="82">         <span class="dt">hspath_typ =</span> <span class="kw">as.factor</span>(hspath_typ),</a>
<a class="sourceLine" id="cb2-83" data-line-number="83">         <span class="dt">tdp_st4 =</span> <span class="kw">as.factor</span>(tdp_st4),</a>
<a class="sourceLine" id="cb2-84" data-line-number="84">        <span class="dt">niareagansc =</span> <span class="kw">ordered</span>(niareagansc, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;4&#39;</span>, <span class="st">&#39;3&#39;</span>, <span class="st">&#39;2&#39;</span>, <span class="st">&#39;1&#39;</span>)), </a>
<a class="sourceLine" id="cb2-85" data-line-number="85">        <span class="dt">CDR =</span> <span class="kw">cut</span>(cts_mmse30_lv, <span class="dt">breaks =</span> <span class="kw">c</span>(<span class="op">-</span><span class="ot">Inf</span>, <span class="dv">11</span>, <span class="dv">21</span>, <span class="dv">26</span>, <span class="dv">30</span>, <span class="ot">Inf</span>), <span class="dt">labels =</span> <span class="kw">c</span>(<span class="dv">3</span>, <span class="dv">2</span>, <span class="dv">1</span>, <span class="fl">0.5</span>, <span class="dv">0</span>), <span class="dt">right =</span> <span class="ot">FALSE</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-86" data-line-number="86"><span class="st">  </span><span class="kw">filter</span>(<span class="op">!</span><span class="kw">is.na</span>(study)) </a>
<a class="sourceLine" id="cb2-87" data-line-number="87"></a>
<a class="sourceLine" id="cb2-88" data-line-number="88"><span class="kw">write_rds</span>(rosmap, <span class="st">&#39;output/rosmap.rds&#39;</span>)</a>
<a class="sourceLine" id="cb2-89" data-line-number="89"></a>
<a class="sourceLine" id="cb2-90" data-line-number="90">df &lt;-<span class="st"> </span>rosmap <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-91" data-line-number="91"><span class="st">    </span><span class="kw">select</span>(study, age_bl, msex, educ, apoe_genotype, cogdx, age_first_ad_dx, Source.Tissue.Type) </a></code></pre></div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-10.1038/sdata.2018.142">
<p>Jager, Philip L De, Yiyi Ma, Cristin McCabe, Jishu Xu, Badri N Vardarajan, Daniel Felsky, Hans-Ulrich Klein, et al. 2018. “A multi-omic atlas of the human frontal cortex for aging and Alzheimer’s disease research.” <em>Scientific Data</em> 5 (1): 180142. <a href="https://doi.org/10.1038/sdata.2018.142">https://doi.org/10.1038/sdata.2018.142</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-5-cohorts.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-1-demographics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["notebook.pdf", "notebook.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
