<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease</title>
  <meta name="description" content="Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  
  <meta name="twitter:description" content="Chapter 4 MSBB | Assessing the causal association of mtDNAcn with Alzheimer’s disease" />
  

<meta name="author" content="Dr. Shea Andrews" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="3-9-neuroimaging.html"/>
<link rel="next" href="4-1-msbb-pathology.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.15/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">mtDNAcn in AD</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html"><i class="fa fa-check"></i><b>1.1</b> Project Updates</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#to-do"><i class="fa fa-check"></i><b>1.1.1</b> To Do</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section"><i class="fa fa-check"></i><b>1.1.2</b> 2020-09-09</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-1"><i class="fa fa-check"></i><b>1.1.3</b> 2020-09-04</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-2"><i class="fa fa-check"></i><b>1.1.4</b> 2020-09-02</a></li>
<li class="chapter" data-level="1.1.5" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-3"><i class="fa fa-check"></i><b>1.1.5</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.6" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-4"><i class="fa fa-check"></i><b>1.1.6</b> 2020-08-22</a></li>
<li class="chapter" data-level="1.1.7" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-5"><i class="fa fa-check"></i><b>1.1.7</b> 2020-08-07</a></li>
<li class="chapter" data-level="1.1.8" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-6"><i class="fa fa-check"></i><b>1.1.8</b> 2020-08-06</a></li>
<li class="chapter" data-level="1.1.9" data-path="1-1-project-updates.html"><a href="1-1-project-updates.html#section-7"><i class="fa fa-check"></i><b>1.1.9</b> 2020-08-05</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html"><i class="fa fa-check"></i><b>1.2</b> Neuropathological Confirmed AD</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#cerad-criteria---1991"><i class="fa fa-check"></i><b>1.2.1</b> CERAD Criteria - 1991</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-reagan-criteria---1997"><i class="fa fa-check"></i><b>1.2.2</b> NIA-Reagan Criteria - 1997</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-2-neuropathological-confirmed-ad.html"><a href="1-2-neuropathological-confirmed-ad.html#nia-aa-criteria---2012"><i class="fa fa-check"></i><b>1.2.3</b> NIA-AA Criteria - 2012</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html"><i class="fa fa-check"></i><b>2.1</b> Mitochondrial DNA</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#adni"><i class="fa fa-check"></i><b>2.1.1</b> ADNI</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#amp-ad"><i class="fa fa-check"></i><b>2.1.2</b> AMP-AD</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#dian"><i class="fa fa-check"></i><b>2.1.3</b> DIAN</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-1-mitochondrial-dna.html"><a href="2-1-mitochondrial-dna.html#calling-mitochondrial-variants-in-amp-ad-dian"><i class="fa fa-check"></i><b>2.1.4</b> Calling Mitochondrial Variants in AMP-AD &amp; DIAN</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html"><i class="fa fa-check"></i><b>2.2</b> Haplogroup Assignment</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-haplogroup-assignment.html"><a href="2-2-haplogroup-assignment.html#haplogrep"><i class="fa fa-check"></i><b>2.2.1</b> Haplogrep</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html"><i class="fa fa-check"></i><b>2.3</b> Estimating mtDNAcn</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-estimating-mtdnacn.html"><a href="2-3-estimating-mtdnacn.html#mosedepth"><i class="fa fa-check"></i><b>2.3.1</b> Mosedepth</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html"><i class="fa fa-check"></i><b>2.4</b> Cell Type Enrichment</a><ul>
<li class="chapter" data-level="2.4.1" data-path="2-4-cell-type-enrichment.html"><a href="2-4-cell-type-enrichment.html#xcell"><i class="fa fa-check"></i><b>2.4.1</b> xCell</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html"><i class="fa fa-check"></i><b>2.5</b> Cohorts</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>2.5.1</b> ROSMAP</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>2.5.2</b> Mayo</a></li>
<li class="chapter" data-level="2.5.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>2.5.3</b> MSBB</a></li>
<li class="chapter" data-level="2.5.4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#amp-ad-cross-study-rnaseq-harmonization"><i class="fa fa-check"></i><b>2.5.4</b> AMP-AD Cross-Study RNAseq Harmonization</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>3</b> ROSMAP</a><ul>
<li class="chapter" data-level="3.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html"><i class="fa fa-check"></i><b>3.1</b> Demographics</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>3.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-2-genetics.html"><a href="3-2-genetics.html"><i class="fa fa-check"></i><b>3.2</b> Genetics</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>3.2.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html"><i class="fa fa-check"></i><b>3.3</b> Mitochondria</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>3.3.1</b> Plots</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.2</b> diagnosis and mtDNAcn</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#blood-cell-counts-and-mtdnacn"><i class="fa fa-check"></i><b>3.3.3</b> Blood cell counts and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html"><i class="fa fa-check"></i><b>3.4</b> RNA-seq</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-4-rna-seq.html"><a href="3-4-rna-seq.html#plots-3"><i class="fa fa-check"></i><b>3.4.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html"><i class="fa fa-check"></i><b>3.5</b> Clinical Diagnosis</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#plots-4"><i class="fa fa-check"></i><b>3.5.1</b> Plots</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>3.5.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-pathology.html"><a href="3-6-pathology.html"><i class="fa fa-check"></i><b>3.6</b> Pathology</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-pathology.html"><a href="3-6-pathology.html#plots-5"><i class="fa fa-check"></i><b>3.6.1</b> Plots</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-pathology.html"><a href="3-6-pathology.html#cross-tabs-1"><i class="fa fa-check"></i><b>3.6.2</b> Cross-Tabs</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="3-7-cognition.html"><a href="3-7-cognition.html"><i class="fa fa-check"></i><b>3.7</b> Cognition</a><ul>
<li class="chapter" data-level="3.7.1" data-path="3-7-cognition.html"><a href="3-7-cognition.html#plots-6"><i class="fa fa-check"></i><b>3.7.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html"><i class="fa fa-check"></i><b>3.8</b> Blood Measures</a><ul>
<li class="chapter" data-level="3.8.1" data-path="3-8-blood-measures.html"><a href="3-8-blood-measures.html#plots-7"><i class="fa fa-check"></i><b>3.8.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="3.9" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html"><i class="fa fa-check"></i><b>3.9</b> Neuroimaging</a><ul>
<li class="chapter" data-level="3.9.1" data-path="3-9-neuroimaging.html"><a href="3-9-neuroimaging.html#plots-8"><i class="fa fa-check"></i><b>3.9.1</b> Plots</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>4</b> MSBB</a><ul>
<li class="chapter" data-level="4.1" data-path="4-1-msbb-pathology.html"><a href="4-1-msbb-pathology.html"><i class="fa fa-check"></i><b>4.1</b> MSBB Pathology</a><ul>
<li class="chapter" data-level="4.1.1" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>4.1.1</b> Cross-tabs</a></li>
<li class="chapter" data-level="4.1.2" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>4.1.2</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-2-msbb-other-variables.html"><a href="4-2-msbb-other-variables.html"><i class="fa fa-check"></i><b>4.2</b> MSBB Other Variables</a></li>
<li class="chapter" data-level="4.3" data-path="4-3-msbb-rna-seq.html"><a href="4-3-msbb-rna-seq.html"><i class="fa fa-check"></i><b>4.3</b> MSBB RNA-seq</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>5</b> MAYO</a><ul>
<li class="chapter" data-level="5.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#demographics"><i class="fa fa-check"></i><b>5.1</b> Demographics</a><ul>
<li class="chapter" data-level="5.1.1" data-path="3-1-demographics.html"><a href="3-1-demographics.html#plots"><i class="fa fa-check"></i><b>5.1.1</b> Plots</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="5-2-diagnosis-and-pathology.html"><a href="5-2-diagnosis-and-pathology.html"><i class="fa fa-check"></i><b>5.2</b> Diagnosis and Pathology</a><ul>
<li class="chapter" data-level="5.2.1" data-path="3-2-genetics.html"><a href="3-2-genetics.html#plots-1"><i class="fa fa-check"></i><b>5.2.1</b> Plots</a></li>
<li class="chapter" data-level="5.2.2" data-path="3-5-clinical-diagnosis.html"><a href="3-5-clinical-diagnosis.html#cross-tabs"><i class="fa fa-check"></i><b>5.2.2</b> Cross-tabs</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-genetics-mitochondria.html"><a href="5-3-genetics-mitochondria.html"><i class="fa fa-check"></i><b>5.3</b> Genetics &amp; Mitochondria</a><ul>
<li class="chapter" data-level="5.3.1" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#plots-2"><i class="fa fa-check"></i><b>5.3.1</b> Plots</a></li>
<li class="chapter" data-level="5.3.2" data-path="3-3-mitochondria.html"><a href="3-3-mitochondria.html#diagnosis-and-mtdnacn"><i class="fa fa-check"></i><b>5.3.2</b> diagnosis and mtDNAcn</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-4-rnaseq.html"><a href="5-4-rnaseq.html"><i class="fa fa-check"></i><b>5.4</b> RNAseq</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-analysis.html"><a href="6-analysis.html"><i class="fa fa-check"></i><b>6</b> Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-statistical-analysis-plan.html"><a href="6-1-statistical-analysis-plan.html"><i class="fa fa-check"></i><b>6.1</b> Statistical Analysis Plan</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html"><i class="fa fa-check"></i><b>6.2</b> Data Wrangling</a><ul>
<li class="chapter" data-level="6.2.1" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#rosmap"><i class="fa fa-check"></i><b>6.2.1</b> ROSMAP</a></li>
<li class="chapter" data-level="6.2.2" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#msbb"><i class="fa fa-check"></i><b>6.2.2</b> MSBB</a></li>
<li class="chapter" data-level="6.2.3" data-path="2-5-cohorts.html"><a href="2-5-cohorts.html#mayo"><i class="fa fa-check"></i><b>6.2.3</b> Mayo</a></li>
<li class="chapter" data-level="6.2.4" data-path="6-2-data-wrangling.html"><a href="6-2-data-wrangling.html#joint"><i class="fa fa-check"></i><b>6.2.4</b> Joint</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html"><i class="fa fa-check"></i><b>6.3</b> Aim 1 (mtHgs)</a><ul>
<li class="chapter" data-level="6.3.1" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1a"><i class="fa fa-check"></i><b>6.3.1</b> Aim 1a</a></li>
<li class="chapter" data-level="6.3.2" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#aim-1b"><i class="fa fa-check"></i><b>6.3.2</b> Aim 1b</a></li>
<li class="chapter" data-level="6.3.3" data-path="6-3-aim-1-mthgs.html"><a href="6-3-aim-1-mthgs.html#sensetivity-analysis"><i class="fa fa-check"></i><b>6.3.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html"><i class="fa fa-check"></i><b>6.4</b> Aim 2 (mtDNAcn)</a><ul>
<li class="chapter" data-level="6.4.1" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2a"><i class="fa fa-check"></i><b>6.4.1</b> Aim 2a</a></li>
<li class="chapter" data-level="6.4.2" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#aim-2b"><i class="fa fa-check"></i><b>6.4.2</b> Aim 2b</a></li>
<li class="chapter" data-level="6.4.3" data-path="6-4-aim-2-mtdnacn.html"><a href="6-4-aim-2-mtdnacn.html#sensetivity-analysis-1"><i class="fa fa-check"></i><b>6.4.3</b> Sensetivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><a href="6-5-assocation-of-mtdnacn-with-clinical-ad-vs-adnc.html"><i class="fa fa-check"></i><b>6.5</b> Assocation of mtDNAcn with Clinical AD vs ADNC</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-aim-3.html"><a href="6-6-aim-3.html"><i class="fa fa-check"></i><b>6.6</b> Aim 3</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-aaic-abstract.html"><a href="7-aaic-abstract.html"><i class="fa fa-check"></i><b>7</b> AAIC Abstract</a></li>
<li class="chapter" data-level="8" data-path="8-final-words.html"><a href="8-final-words.html"><i class="fa fa-check"></i><b>8</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Assessing the causal association of mtDNAcn with Alzheimer’s disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="msbb" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> MSBB</h1>
<p>Samples come from 364 postmortem control, mild cognitive impaired (MCI) and AD brains with rich clinical and pathophysiological data from the Mount Sinai/JJ Peters VA Medical Center Brain Bank (MSBB–Mount Sinai NIH Neurobiobank) cohort. The majority (301) of the samples were of European ancestry, while 36 were African American, 25 were Latino, one was Asian, and one was unknown for race. Neuropathological assessments were performed using CERAD scores and Braak Staging. The CDR scale was conducted for assessment of dementia and cognitive status (<span class="citation">Wang et al. (<a href="#ref-10.1038/sdata.2018.185">2018</a>)</span>).</p>
<p>Clinical Code Book: <a href="https://adknowledgeportal.synapse.org/Explore/Studies?Study=syn3159438">Synapse</a></p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" data-line-number="1"><span class="co">## MSBB</span></a>
<a class="sourceLine" id="cb22-2" data-line-number="2">msbb.raw1 &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&#39;data/AMPAD/msbb/WGS_Metadata.txt&#39;</span>) </a>
<a class="sourceLine" id="cb22-3" data-line-number="3">msbb.raw2 &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&#39;data/AMPAD_extra/msbb.wgs.meta.tsv&#39;</span>)</a>
<a class="sourceLine" id="cb22-4" data-line-number="4">msbb.path &lt;-<span class="st"> </span>readxl<span class="op">::</span><span class="kw">read_xlsx</span>(<span class="st">&#39;data/AMPAD_extra/msbb/TempAmp_AD-Shea-2-2020.xlsx&#39;</span>, <span class="dt">sheet =</span> <span class="dv">2</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb22-5" data-line-number="5"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">id =</span> <span class="kw">as.character</span>(id))</a>
<a class="sourceLine" id="cb22-6" data-line-number="6">msbb.raw &lt;-<span class="st"> </span>msbb.raw1 <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb22-7" data-line-number="7"><span class="st">  </span><span class="kw">select</span>(WGS, individualIdentifier, PMI, RACE, CDR, SEX, NP<span class="fl">.1</span>, PlaqueMean, bbscore) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb22-8" data-line-number="8"><span class="st">  </span><span class="kw">left_join</span>(<span class="kw">select</span>(msbb.raw2, Libid, AOD, APOE_inferred), <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&#39;WGS&#39;</span> =<span class="st"> &#39;Libid&#39;</span>)) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb22-9" data-line-number="9"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">SubNum =</span> <span class="kw">str_extract</span>(<span class="kw">gsub</span>(<span class="st">&quot;(?&lt;![0-9])0+&quot;</span>, <span class="st">&quot;&quot;</span>, individualIdentifier, <span class="dt">perl =</span> <span class="ot">TRUE</span>), <span class="st">&#39;[:digit:].*&#39;</span>)) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb22-10" data-line-number="10"><span class="st">  </span><span class="kw">left_join</span>(msbb.path, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&#39;SubNum&#39;</span> =<span class="st"> &#39;id&#39;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb22-11" data-line-number="11"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">study =</span> <span class="st">&#39;MSBB&#39;</span>, </a>
<a class="sourceLine" id="cb22-12" data-line-number="12">         <span class="dt">WGS =</span> <span class="kw">as.character</span>(WGS))</a>
<a class="sourceLine" id="cb22-13" data-line-number="13"></a>
<a class="sourceLine" id="cb22-14" data-line-number="14">mosdepth &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&quot;data/mosdepth/mosdepth_mtDNAcn_All.txt&quot;</span>)</a>
<a class="sourceLine" id="cb22-15" data-line-number="15">haplogrep &lt;-<span class="st"> </span><span class="kw">read_tsv</span>(<span class="st">&quot;data/haplogrep/haplogrep_jointAll.txt&quot;</span>)</a>
<a class="sourceLine" id="cb22-16" data-line-number="16"></a>
<a class="sourceLine" id="cb22-17" data-line-number="17">xcell.raw &lt;-<span class="st"> </span><span class="kw">read_csv</span>(<span class="st">&quot;data/xcell/ampad_xCell.csv&quot;</span>) </a></code></pre></div>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb23-1" data-line-number="1"><span class="co">## Wrangle xcell data</span></a>
<a class="sourceLine" id="cb23-2" data-line-number="2">xcell &lt;-<span class="st"> </span>xcell.raw <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-3" data-line-number="3"><span class="st">  </span><span class="kw">filter</span>(study <span class="op">==</span><span class="st"> &quot;MSBB&quot;</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-4" data-line-number="4"><span class="st">  </span><span class="kw">group_by</span>(Tissue, ID) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-5" data-line-number="5"><span class="st">  </span><span class="kw">slice</span>(<span class="kw">which.max</span>(<span class="kw">str_length</span>(SampleID))) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-6" data-line-number="6"><span class="st">  </span><span class="kw">ungroup</span>() <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-7" data-line-number="7"><span class="st">  </span><span class="kw">select</span>(<span class="op">-</span>SampleID, <span class="op">-</span>study, <span class="op">-</span>Neurons.pval, <span class="op">-</span>Astrocytes.pval) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-8" data-line-number="8"><span class="st">  </span><span class="kw">rename</span>(<span class="dt">rna_seq_tissue =</span> Tissue, <span class="dt">rna_seq_batch =</span> batch) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb23-9" data-line-number="9"><span class="st">  </span><span class="kw">pivot_wider</span>(<span class="dt">names_from =</span> rna_seq_tissue, <span class="dt">values_from =</span> <span class="kw">c</span>(Neurons, Astrocytes, rna_seq_batch, RIN))</a>
<a class="sourceLine" id="cb23-10" data-line-number="10"></a>
<a class="sourceLine" id="cb23-11" data-line-number="11"><span class="co">## Recode NP.1 -&gt; ceradsc to be consistent with ROSMAP</span></a>
<a class="sourceLine" id="cb23-12" data-line-number="12">msbb &lt;-<span class="st"> </span>msbb.raw <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-13" data-line-number="13"><span class="st">  </span><span class="kw">left_join</span>(haplogrep, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&#39;WGS&#39;</span> =<span class="st"> &#39;SampleID&#39;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-14" data-line-number="14"><span class="st">  </span><span class="kw">left_join</span>(mosdepth, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&#39;WGS&#39;</span> =<span class="st"> &#39;SampleID&#39;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-15" data-line-number="15"><span class="st">  </span><span class="kw">left_join</span>(xcell, <span class="dt">by =</span> <span class="kw">c</span>(<span class="st">&quot;individualIdentifier&quot;</span> =<span class="st"> &quot;ID&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-16" data-line-number="16"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">id =</span> <span class="kw">paste0</span>(<span class="st">&#39;MSBB&#39;</span>, WGS)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-17" data-line-number="17"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">APOE_inferred =</span> <span class="kw">recode</span>(APOE_inferred, <span class="st">&#39;e2/e2&#39;</span> =<span class="st"> &#39;22&#39;</span>, <span class="st">&#39;e2/e3&#39;</span> =<span class="st"> &#39;23&#39;</span>, <span class="st">&#39;e3/e3&#39;</span> =<span class="st"> &#39;33&#39;</span>, <span class="st">&#39;e2/e4&#39;</span> =<span class="st"> &#39;24&#39;</span>, <span class="st">&#39;e3/e4&#39;</span> =<span class="st"> &#39;34&#39;</span>, <span class="st">&#39;e4/e4&#39;</span> =<span class="st"> &#39;44&#39;</span>),</a>
<a class="sourceLine" id="cb23-18" data-line-number="18">        <span class="dt">apoe4 =</span> <span class="kw">recode</span>(APOE_inferred, <span class="st">&#39;22&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;23&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;33&#39;</span> =<span class="st"> &#39;e4-&#39;</span>, <span class="st">&#39;24&#39;</span> =<span class="st"> &#39;e4+&#39;</span>, <span class="st">&#39;34&#39;</span> =<span class="st"> &#39;e4+&#39;</span>, <span class="st">&#39;44&#39;</span> =<span class="st"> &#39;e4+&#39;</span>),</a>
<a class="sourceLine" id="cb23-19" data-line-number="19">        <span class="dt">SEX =</span> <span class="kw">as.factor</span>(SEX),</a>
<a class="sourceLine" id="cb23-20" data-line-number="20">        <span class="dt">aod_cat =</span> <span class="kw">cut</span>(AOD, <span class="kw">c</span>(<span class="dv">50</span>, <span class="dv">60</span>, <span class="dv">70</span>, <span class="dv">80</span>, <span class="dv">90</span>, <span class="ot">Inf</span>), <span class="kw">c</span>(<span class="st">&#39;50-59&#39;</span>, <span class="st">&#39;60-69&#39;</span>, <span class="st">&#39;70-79&#39;</span>, <span class="st">&#39;80-89&#39;</span>, <span class="st">&#39;90+&#39;</span>), <span class="dt">right =</span> <span class="ot">FALSE</span>),</a>
<a class="sourceLine" id="cb23-21" data-line-number="21">        <span class="dt">aod_cat =</span> <span class="kw">ordered</span>(aod_cat, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;50-59&#39;</span>, <span class="st">&#39;60-69&#39;</span>, <span class="st">&#39;70-79&#39;</span>, <span class="st">&#39;80-89&#39;</span>, <span class="st">&#39;90+&#39;</span>)), </a>
<a class="sourceLine" id="cb23-22" data-line-number="22">        <span class="dt">SourceTissue =</span> <span class="st">&#39;prefrontal cortex&#39;</span>,</a>
<a class="sourceLine" id="cb23-23" data-line-number="23">        <span class="dt">APOE_inferred =</span> <span class="kw">as.factor</span>(APOE_inferred), </a>
<a class="sourceLine" id="cb23-24" data-line-number="24">        <span class="dt">apoe4 =</span> <span class="kw">as.factor</span>(apoe4), </a>
<a class="sourceLine" id="cb23-25" data-line-number="25">        <span class="dt">RACE =</span> <span class="kw">as.factor</span>(RACE), </a>
<a class="sourceLine" id="cb23-26" data-line-number="26">        <span class="dt">z_mtdnacn =</span> <span class="kw">scale</span>(mtcn_avg, <span class="dt">center =</span> <span class="ot">TRUE</span>, <span class="dt">scale =</span> <span class="ot">TRUE</span>)[,<span class="dv">1</span>],</a>
<a class="sourceLine" id="cb23-27" data-line-number="27">        <span class="dt">macro =</span> <span class="kw">case_when</span>(</a>
<a class="sourceLine" id="cb23-28" data-line-number="28">          <span class="kw">str_detect</span>(Haplogroup, <span class="st">&quot;^L|^HV|^JT&quot;</span>) <span class="op">~</span><span class="st"> </span><span class="kw">substr</span>(Haplogroup, <span class="dt">start =</span> <span class="dv">1</span>, <span class="dt">stop =</span> <span class="dv">2</span>),</a>
<a class="sourceLine" id="cb23-29" data-line-number="29">                     <span class="ot">TRUE</span> <span class="op">~</span><span class="st"> </span><span class="kw">substr</span>(Haplogroup, <span class="dt">start =</span> <span class="dv">1</span>, <span class="dt">stop =</span> <span class="dv">1</span>)),</a>
<a class="sourceLine" id="cb23-30" data-line-number="30">        <span class="dt">cerad =</span> <span class="kw">recode</span>(NP<span class="fl">.1</span>, <span class="st">&#39;1&#39;</span> =<span class="st"> &#39;Normal&#39;</span>, <span class="st">&#39;2&#39;</span> =<span class="st"> &#39;Definite&#39;</span>, <span class="st">&#39;3&#39;</span> =<span class="st"> &#39;Probable&#39;</span>, <span class="st">&#39;4&#39;</span> =<span class="st"> &#39;Possible&#39;</span>),</a>
<a class="sourceLine" id="cb23-31" data-line-number="31">        <span class="dt">cerad =</span> <span class="kw">ordered</span>(cerad, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;Normal&#39;</span>, <span class="st">&#39;Possible&#39;</span>, <span class="st">&#39;Probable&#39;</span>, <span class="st">&#39;Definite&#39;</span>)),</a>
<a class="sourceLine" id="cb23-32" data-line-number="32">        <span class="dt">ceradsc =</span> <span class="kw">pmax</span>(HippoPlaquesWCoresValue, EntorPlaquesWCoresValue, MidPlaquesWCoresValue, </a>
<a class="sourceLine" id="cb23-33" data-line-number="33">                       SupPlaquesWCoresValue, InfPlaquesWCoresValue, OcciPlaquesWCoresValue),</a>
<a class="sourceLine" id="cb23-34" data-line-number="34">        <span class="dt">ceradsc =</span> <span class="kw">ordered</span>(ceradsc),</a>
<a class="sourceLine" id="cb23-35" data-line-number="35">        <span class="dt">ceradsc =</span> <span class="kw">fct_recode</span>(ceradsc, <span class="st">&#39;4&#39;</span> =<span class="st"> &#39;0&#39;</span>, <span class="st">&#39;3&#39;</span> =<span class="st"> &#39;1&#39;</span>, <span class="st">&#39;2&#39;</span> =<span class="st"> &#39;3&#39;</span>, <span class="st">&#39;1&#39;</span> =<span class="st"> &#39;5&#39;</span>),</a>
<a class="sourceLine" id="cb23-36" data-line-number="36">        <span class="dt">bbscore =</span> <span class="kw">ordered</span>(bbscore, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;0&#39;</span>, <span class="st">&#39;1&#39;</span>, <span class="st">&#39;2&#39;</span>, <span class="st">&#39;3&#39;</span>, <span class="st">&#39;4&#39;</span>, <span class="st">&#39;5&#39;</span>, <span class="st">&#39;6&#39;</span>)),</a>
<a class="sourceLine" id="cb23-37" data-line-number="37">        <span class="dt">niareagansc =</span> <span class="kw">case_when</span>(</a>
<a class="sourceLine" id="cb23-38" data-line-number="38">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">4</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">==</span><span class="st"> </span><span class="dv">0</span> <span class="op">~</span><span class="st"> </span><span class="dv">4</span>, </a>
<a class="sourceLine" id="cb23-39" data-line-number="39">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">4</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">1</span><span class="op">:</span><span class="dv">6</span>) <span class="op">~</span><span class="st"> </span><span class="dv">3</span>, </a>
<a class="sourceLine" id="cb23-40" data-line-number="40">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">3</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">0</span><span class="op">:</span><span class="dv">6</span>) <span class="op">~</span><span class="st"> </span><span class="dv">3</span>, </a>
<a class="sourceLine" id="cb23-41" data-line-number="41">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">2</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">0</span><span class="op">:</span><span class="dv">2</span>) <span class="op">~</span><span class="st"> </span><span class="dv">3</span>,</a>
<a class="sourceLine" id="cb23-42" data-line-number="42">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">2</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">3</span><span class="op">:</span><span class="dv">6</span>) <span class="op">~</span><span class="st"> </span><span class="dv">2</span>,</a>
<a class="sourceLine" id="cb23-43" data-line-number="43">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">1</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">0</span><span class="op">:</span><span class="dv">4</span>) <span class="op">~</span><span class="st"> </span><span class="dv">2</span>,</a>
<a class="sourceLine" id="cb23-44" data-line-number="44">          ceradsc <span class="op">==</span><span class="st"> </span><span class="dv">1</span> <span class="op">&amp;</span><span class="st"> </span>bbscore <span class="op">%in%</span><span class="st"> </span><span class="kw">c</span>(<span class="dv">5</span><span class="op">:</span><span class="dv">6</span>) <span class="op">~</span><span class="st"> </span><span class="dv">1</span>,</a>
<a class="sourceLine" id="cb23-45" data-line-number="45">        ), </a>
<a class="sourceLine" id="cb23-46" data-line-number="46">        <span class="dt">niareagansc =</span> <span class="kw">ordered</span>(niareagansc, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&#39;4&#39;</span>, <span class="st">&#39;3&#39;</span>, <span class="st">&#39;2&#39;</span>, <span class="st">&#39;1&#39;</span>)),</a>
<a class="sourceLine" id="cb23-47" data-line-number="47">        <span class="dt">ad_reagan =</span> <span class="kw">fct_recode</span>(niareagansc, <span class="st">&quot;1&quot;</span> =<span class="st"> &quot;1&quot;</span>, <span class="st">&quot;1&quot;</span> =<span class="st"> &quot;2&quot;</span>, <span class="st">&quot;0&quot;</span> =<span class="st"> &quot;3&quot;</span>, <span class="st">&quot;0&quot;</span> =<span class="st"> &quot;4&quot;</span>),</a>
<a class="sourceLine" id="cb23-48" data-line-number="48">        <span class="dt">braaksc_B =</span> <span class="kw">fct_recode</span>(bbscore, <span class="dt">B0 =</span> <span class="st">&quot;0&quot;</span>, <span class="dt">B1 =</span> <span class="st">&quot;1&quot;</span>, <span class="dt">B1 =</span> <span class="st">&quot;2&quot;</span>, <span class="dt">B2 =</span> <span class="st">&quot;3&quot;</span>, <span class="dt">B2 =</span> <span class="st">&quot;4&quot;</span>, <span class="dt">B3 =</span> <span class="st">&quot;5&quot;</span>, <span class="dt">B3 =</span> <span class="st">&quot;6&quot;</span>), </a>
<a class="sourceLine" id="cb23-49" data-line-number="49">        <span class="dt">ceradsc_C =</span> <span class="kw">fct_recode</span>(ceradsc, <span class="dt">C0 =</span> <span class="st">&quot;1&quot;</span>, <span class="dt">C1 =</span> <span class="st">&quot;2&quot;</span>, <span class="dt">C2 =</span> <span class="st">&quot;3&quot;</span>, <span class="dt">C3 =</span> <span class="st">&quot;4&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-50" data-line-number="50"><span class="st">  </span><span class="kw">mutate_at</span>(<span class="kw">vars</span>(HippoPlaquesWCoresValue, EntorPlaquesWCoresValue, MidPlaquesWCoresValue, </a>
<a class="sourceLine" id="cb23-51" data-line-number="51">                       SupPlaquesWCoresValue, InfPlaquesWCoresValue, OcciPlaquesWCoresValue), </a>
<a class="sourceLine" id="cb23-52" data-line-number="52">            ordered) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb23-53" data-line-number="53"><span class="st">  </span><span class="kw">select</span>(id, individualIdentifier, study, <span class="dt">sex =</span> SEX, <span class="dt">race =</span> RACE, SourceTissue, </a>
<a class="sourceLine" id="cb23-54" data-line-number="54">         PMI, NP<span class="fl">.1</span>, cerad, niareagansc, ad_reagan, ceradsc, HippoPlaquesWCoresValue, </a>
<a class="sourceLine" id="cb23-55" data-line-number="55">         EntorPlaquesWCoresValue, MidPlaquesWCoresValue, SupPlaquesWCoresValue, </a>
<a class="sourceLine" id="cb23-56" data-line-number="56">         InfPlaquesWCoresValue, OcciPlaquesWCoresValue, <span class="dt">braaksc =</span> bbscore, braaksc_B, </a>
<a class="sourceLine" id="cb23-57" data-line-number="57">         ceradsc_C, <span class="dt">apoe =</span> APOE_inferred, apoe4, <span class="dt">aod =</span> AOD, aod_cat, <span class="dt">cdr =</span> CDR, </a>
<a class="sourceLine" id="cb23-58" data-line-number="58">         PlaqueMean, mtcn_avg, z_mtdnacn, Haplogroup, Quality, macro, </a>
<a class="sourceLine" id="cb23-59" data-line-number="59">         Neurons_BM10, Neurons_BM22, Neurons_BM36, Neurons_BM44, </a>
<a class="sourceLine" id="cb23-60" data-line-number="60">         Astrocytes_BM10, Astrocytes_BM22, Astrocytes_BM36, Astrocytes_BM44, </a>
<a class="sourceLine" id="cb23-61" data-line-number="61">         rna_seq_batch_BM10, rna_seq_batch_BM22, rna_seq_batch_BM36, rna_seq_batch_BM44, </a>
<a class="sourceLine" id="cb23-62" data-line-number="62">         RIN_BM10, RIN_BM22, RIN_BM36, RIN_BM44)</a>
<a class="sourceLine" id="cb23-63" data-line-number="63"></a>
<a class="sourceLine" id="cb23-64" data-line-number="64"><span class="kw">write_rds</span>(msbb, <span class="st">&#39;output/msbb.rds&#39;</span>)</a></code></pre></div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-10.1038/sdata.2018.185">
<p>Wang, Minghui, Noam D. Beckmann, Panos Roussos, Erming Wang, Xianxiao Zhou, Qian Wang, Chen Ming, et al. 2018. “The Mount Sinai cohort of large-scale genomic, transcriptomic and proteomic data in Alzheimer’s disease.” <em>Scientific Data</em> 5 (1): 180185. <a href="https://doi.org/10.1038/sdata.2018.185">https://doi.org/10.1038/sdata.2018.185</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-9-neuroimaging.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-1-msbb-pathology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["notebook.pdf", "notebook.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
